2020-05-05 22:40:27.757322 (MainThread): Running with dbt=0.16.0
2020-05-05 22:40:27.888638 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, exclude=None, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='/Users/kristin/.dbt', project_dir=None, record_timing_info=None, rpc_method='run', single_threaded=False, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2020-05-05 22:40:27.889852 (MainThread): Tracking: tracking
2020-05-05 22:40:27.895215 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1120d9b90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x111c05890>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x11210ea50>]}
2020-05-05 22:40:27.912314 (MainThread): Partial parsing not enabled
2020-05-05 22:40:27.913973 (MainThread): Parsing macros/core.sql
2020-05-05 22:40:27.917690 (MainThread): Parsing macros/materializations/helpers.sql
2020-05-05 22:40:27.924820 (MainThread): Parsing macros/materializations/snapshot/snapshot_merge.sql
2020-05-05 22:40:27.926413 (MainThread): Parsing macros/materializations/snapshot/strategies.sql
2020-05-05 22:40:27.942512 (MainThread): Parsing macros/materializations/snapshot/snapshot.sql
2020-05-05 22:40:27.974101 (MainThread): Parsing macros/materializations/seed/seed.sql
2020-05-05 22:40:27.993909 (MainThread): Parsing macros/materializations/incremental/helpers.sql
2020-05-05 22:40:27.995651 (MainThread): Parsing macros/materializations/incremental/incremental.sql
2020-05-05 22:40:28.001636 (MainThread): Parsing macros/materializations/common/merge.sql
2020-05-05 22:40:28.013141 (MainThread): Parsing macros/materializations/table/table.sql
2020-05-05 22:40:28.019300 (MainThread): Parsing macros/materializations/view/view.sql
2020-05-05 22:40:28.024978 (MainThread): Parsing macros/materializations/view/create_or_replace_view.sql
2020-05-05 22:40:28.029487 (MainThread): Parsing macros/etc/get_custom_alias.sql
2020-05-05 22:40:28.030359 (MainThread): Parsing macros/etc/query.sql
2020-05-05 22:40:28.031343 (MainThread): Parsing macros/etc/is_incremental.sql
2020-05-05 22:40:28.032871 (MainThread): Parsing macros/etc/get_relation_comment.sql
2020-05-05 22:40:28.034785 (MainThread): Parsing macros/etc/datetime.sql
2020-05-05 22:40:28.042995 (MainThread): Parsing macros/etc/get_custom_schema.sql
2020-05-05 22:40:28.044893 (MainThread): Parsing macros/etc/get_custom_database.sql
2020-05-05 22:40:28.045862 (MainThread): Parsing macros/adapters/common.sql
2020-05-05 22:40:28.083468 (MainThread): Parsing macros/schema_tests/relationships.sql
2020-05-05 22:40:28.084524 (MainThread): Parsing macros/schema_tests/not_null.sql
2020-05-05 22:40:28.085377 (MainThread): Parsing macros/schema_tests/unique.sql
2020-05-05 22:40:28.086381 (MainThread): Parsing macros/schema_tests/accepted_values.sql
2020-05-05 22:40:28.088531 (MainThread): Parsing macros/etc.sql
2020-05-05 22:40:28.089153 (MainThread): Parsing macros/catalog.sql
2020-05-05 22:40:28.095543 (MainThread): Parsing macros/adapters.sql
2020-05-05 22:40:28.114763 (MainThread): Parsing macros/materializations/seed.sql
2020-05-05 22:40:28.116593 (MainThread): Parsing macros/materializations/view.sql
2020-05-05 22:40:28.117929 (MainThread): Parsing macros/materializations/table.sql
2020-05-05 22:40:28.127043 (MainThread): Parsing macros/materializations/incremental.sql
2020-05-05 22:40:28.138752 (MainThread): Parsing macros/materializations/snapshot.sql
2020-05-05 22:40:28.154515 (MainThread): Partial parsing not enabled
2020-05-05 22:40:28.182362 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-05-05 22:40:28.182452 (MainThread): Opening a new connection, currently in state init
2020-05-05 22:40:28.199033 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-05-05 22:40:28.199122 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:40:28.203767 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-05-05 22:40:28.203841 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:40:28.208530 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-05-05 22:40:28.208604 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:40:28.236828 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_account_account_id".
2020-05-05 22:40:28.236904 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:40:28.244756 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_account_account_id".
2020-05-05 22:40:28.244832 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:40:28.252211 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:40:28.252285 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:40:28.258104 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:40:28.258177 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:40:28.264329 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_user_id".
2020-05-05 22:40:28.264402 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:40:28.270346 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_user_id".
2020-05-05 22:40:28.270419 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:40:28.276349 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_role_user_role_id".
2020-05-05 22:40:28.276422 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:40:28.282359 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_role_user_role_id".
2020-05-05 22:40:28.282435 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:40:28.315529 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1122cc090>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1122ccbd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x112344790>]}
2020-05-05 22:40:28.315688 (MainThread): Flushing usage events
2020-05-05 22:40:28.661760 (MainThread): Connection 'test.salesforce_source.unique_stg_salesforce_user_role_user_role_id' was properly closed.
2020-05-05 22:40:28.662002 (MainThread): Encountered an error:
2020-05-05 22:40:28.662157 (MainThread): Compilation Error
  Invalid sources config given in models/base/src_salesforce.yml @ sources: {'name': 'salesforce', 'loader': 'fivetran', 'loaded_at_field': '_fivetran_synced', 'freshness': {'warn_after': {'count': 24, 'period': 'hour'}, 'error_after': {'count': 48, 'period': 'hour'}}, 'tables': [{'name': 'account', 'loaded_at_field': '_fivetran_synced', 'description': 'Represents an individual account, which is an organization or person involved with your business (such as customers, competitors, and partners).', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation'}, {'name': 'name', 'description': 'Name of the account. Maximum size is 255 characters.'}, {'name': 'industry', 'description': 'An industry associated with this account. Maximum size is 40 characters.'}, {'name': 'number_of_employees', 'description': 'Number of employees working at the company represented by this account. Maximum size is eight digits.'}, {'name': 'annual_revenue', 'description': 'Estimated annual revenue of the account.'}]}, {'name': 'opportunity', 'loaded_at_field': '_fivetran_synced', 'description': 'Represents an opportunity, which is a sale or pending deal.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'amount', 'description': 'Estimated total sale amount. For opportunities with products, the amount is the sum of the related products.'}, {'name': 'probability', 'description': None}, {'name': 'created_date', 'description': 'The system generated time stamp of when the opportunity was created.'}, {'name': 'is_won', 'description': 'Boolean for if the opportunity is won.  Controled by stage_name.'}, {'name': 'is_closed', 'description': 'Boolean for if the opportunity is closed.  Controled by stage_name.'}, {'name': 'forecast_category', 'description': "The category of the state of the opportunity.  Default values are 'pipeline','forecast','bestcase','closed', and 'omited'"}, {'name': 'owner_id', 'description': 'ID of the User who has been assigned to work this opportunity.'}, {'name': 'close_date', 'description': 'Date when the opportunity has been, or is expected to close.'}, {'name': 'stage_name', 'description': 'Current stage of this record. The StageName field controls several other fields on an opportunity.'}]}, {'name': 'user', 'freshness': None, 'description': 'Represents a user in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'name', 'description': 'Concatenation of FirstName and LastName fields.'}, {'name': 'city', 'description': 'The city associated with the user.'}, {'name': 'state', 'description': 'The state associated with the user.'}, {'name': 'manager_id', 'description': 'The Id of the user who manages this user.'}, {'name': 'user_role_id', 'description': 'ID of the user’s user_role'}]}, {'name': 'user_role', 'freshness': None, 'description': 'Represents a user role in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation'}, {'name': 'name', 'description': 'The name assigned the the role'}]}]} - at path ['tables'][1]['columns'][2]['description']: None is not of type 'string'
2020-05-05 22:40:28.664704 (MainThread): jsonschema.exceptions.ValidationError: None is not of type 'string'

Failed validating 'type' in schema['properties']['tables']['items']['properties']['columns']['items']['properties']['description']:
    {'default': '', 'type': 'string'}

On instance['tables'][1]['columns'][2]['description']:
    None

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 499, in get_unparsed_target
    source = UnparsedSourceDefinition.from_dict(data)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/hologram/__init__.py", line 594, in from_dict
    cls.validate(data)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/hologram/__init__.py", line 936, in validate
    raise ValidationError.create_from(error) from error
hologram.ValidationError: None is not of type 'string'

Failed validating 'type' in schema['properties']['tables']['items']['properties']['columns']['items']['properties']['description']:
    {'default': '', 'type': 'string'}

On instance['tables'][1]['columns'][2]['description']:
    None

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/main.py", line 81, in main
    results, succeeded = handle_and_check(args)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/main.py", line 159, in handle_and_check
    task, res = run_from_args(parsed)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/main.py", line 212, in run_from_args
    results = task.run()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 351, in run
    self._runtime_initialize()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 107, in _runtime_initialize
    super()._runtime_initialize()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 75, in _runtime_initialize
    self.load_manifest()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 63, in load_manifest
    self.manifest = get_full_manifest(self.config)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/perf_utils.py", line 23, in get_full_manifest
    return load_manifest(config, internal, set_header)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 646, in load_manifest
    return ManifestLoader.load_all(config, internal_manifest, macro_hook)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 336, in load_all
    loader.load(internal_manifest=internal_manifest)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 208, in load
    self.parse_project(project, macro_manifest, old_results)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 182, in parse_project
    self.parse_with_cache(path, parser, old_results)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 138, in parse_with_cache
    parser.parse_file(block)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 302, in parse_file
    for test_block in parser.parse():
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 394, in parse
    for node in self.get_unparsed_target():
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 502, in get_unparsed_target
    raise CompilationException(msg) from exc
dbt.exceptions.CompilationException: Compilation Error
  Invalid sources config given in models/base/src_salesforce.yml @ sources: {'name': 'salesforce', 'loader': 'fivetran', 'loaded_at_field': '_fivetran_synced', 'freshness': {'warn_after': {'count': 24, 'period': 'hour'}, 'error_after': {'count': 48, 'period': 'hour'}}, 'tables': [{'name': 'account', 'loaded_at_field': '_fivetran_synced', 'description': 'Represents an individual account, which is an organization or person involved with your business (such as customers, competitors, and partners).', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation'}, {'name': 'name', 'description': 'Name of the account. Maximum size is 255 characters.'}, {'name': 'industry', 'description': 'An industry associated with this account. Maximum size is 40 characters.'}, {'name': 'number_of_employees', 'description': 'Number of employees working at the company represented by this account. Maximum size is eight digits.'}, {'name': 'annual_revenue', 'description': 'Estimated annual revenue of the account.'}]}, {'name': 'opportunity', 'loaded_at_field': '_fivetran_synced', 'description': 'Represents an opportunity, which is a sale or pending deal.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'amount', 'description': 'Estimated total sale amount. For opportunities with products, the amount is the sum of the related products.'}, {'name': 'probability', 'description': None}, {'name': 'created_date', 'description': 'The system generated time stamp of when the opportunity was created.'}, {'name': 'is_won', 'description': 'Boolean for if the opportunity is won.  Controled by stage_name.'}, {'name': 'is_closed', 'description': 'Boolean for if the opportunity is closed.  Controled by stage_name.'}, {'name': 'forecast_category', 'description': "The category of the state of the opportunity.  Default values are 'pipeline','forecast','bestcase','closed', and 'omited'"}, {'name': 'owner_id', 'description': 'ID of the User who has been assigned to work this opportunity.'}, {'name': 'close_date', 'description': 'Date when the opportunity has been, or is expected to close.'}, {'name': 'stage_name', 'description': 'Current stage of this record. The StageName field controls several other fields on an opportunity.'}]}, {'name': 'user', 'freshness': None, 'description': 'Represents a user in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'name', 'description': 'Concatenation of FirstName and LastName fields.'}, {'name': 'city', 'description': 'The city associated with the user.'}, {'name': 'state', 'description': 'The state associated with the user.'}, {'name': 'manager_id', 'description': 'The Id of the user who manages this user.'}, {'name': 'user_role_id', 'description': 'ID of the user’s user_role'}]}, {'name': 'user_role', 'freshness': None, 'description': 'Represents a user role in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation'}, {'name': 'name', 'description': 'The name assigned the the role'}]}]} - at path ['tables'][1]['columns'][2]['description']: None is not of type 'string'

2020-05-05 22:45:32.403830 (MainThread): Running with dbt=0.16.0
2020-05-05 22:45:32.548850 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, exclude=None, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='/Users/kristin/.dbt', project_dir=None, record_timing_info=None, rpc_method='run', single_threaded=False, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2020-05-05 22:45:32.549727 (MainThread): Tracking: tracking
2020-05-05 22:45:32.555885 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10931cad0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109832a10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109832150>]}
2020-05-05 22:45:32.573099 (MainThread): Partial parsing not enabled
2020-05-05 22:45:32.574820 (MainThread): Parsing macros/core.sql
2020-05-05 22:45:32.578597 (MainThread): Parsing macros/materializations/helpers.sql
2020-05-05 22:45:32.585845 (MainThread): Parsing macros/materializations/snapshot/snapshot_merge.sql
2020-05-05 22:45:32.587463 (MainThread): Parsing macros/materializations/snapshot/strategies.sql
2020-05-05 22:45:32.603266 (MainThread): Parsing macros/materializations/snapshot/snapshot.sql
2020-05-05 22:45:32.633460 (MainThread): Parsing macros/materializations/seed/seed.sql
2020-05-05 22:45:32.652391 (MainThread): Parsing macros/materializations/incremental/helpers.sql
2020-05-05 22:45:32.654078 (MainThread): Parsing macros/materializations/incremental/incremental.sql
2020-05-05 22:45:32.659668 (MainThread): Parsing macros/materializations/common/merge.sql
2020-05-05 22:45:32.670890 (MainThread): Parsing macros/materializations/table/table.sql
2020-05-05 22:45:32.677006 (MainThread): Parsing macros/materializations/view/view.sql
2020-05-05 22:45:32.682677 (MainThread): Parsing macros/materializations/view/create_or_replace_view.sql
2020-05-05 22:45:32.688838 (MainThread): Parsing macros/etc/get_custom_alias.sql
2020-05-05 22:45:32.690120 (MainThread): Parsing macros/etc/query.sql
2020-05-05 22:45:32.691213 (MainThread): Parsing macros/etc/is_incremental.sql
2020-05-05 22:45:32.693439 (MainThread): Parsing macros/etc/get_relation_comment.sql
2020-05-05 22:45:32.695694 (MainThread): Parsing macros/etc/datetime.sql
2020-05-05 22:45:32.706198 (MainThread): Parsing macros/etc/get_custom_schema.sql
2020-05-05 22:45:32.708713 (MainThread): Parsing macros/etc/get_custom_database.sql
2020-05-05 22:45:32.710105 (MainThread): Parsing macros/adapters/common.sql
2020-05-05 22:45:32.749479 (MainThread): Parsing macros/schema_tests/relationships.sql
2020-05-05 22:45:32.750587 (MainThread): Parsing macros/schema_tests/not_null.sql
2020-05-05 22:45:32.751452 (MainThread): Parsing macros/schema_tests/unique.sql
2020-05-05 22:45:32.752474 (MainThread): Parsing macros/schema_tests/accepted_values.sql
2020-05-05 22:45:32.754641 (MainThread): Parsing macros/etc.sql
2020-05-05 22:45:32.755264 (MainThread): Parsing macros/catalog.sql
2020-05-05 22:45:32.761775 (MainThread): Parsing macros/adapters.sql
2020-05-05 22:45:32.781294 (MainThread): Parsing macros/materializations/seed.sql
2020-05-05 22:45:32.783154 (MainThread): Parsing macros/materializations/view.sql
2020-05-05 22:45:32.784511 (MainThread): Parsing macros/materializations/table.sql
2020-05-05 22:45:32.793789 (MainThread): Parsing macros/materializations/incremental.sql
2020-05-05 22:45:32.805690 (MainThread): Parsing macros/materializations/snapshot.sql
2020-05-05 22:45:32.822102 (MainThread): Partial parsing not enabled
2020-05-05 22:45:32.850484 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-05-05 22:45:32.850578 (MainThread): Opening a new connection, currently in state init
2020-05-05 22:45:32.867161 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-05-05 22:45:32.867240 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:45:32.871768 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-05-05 22:45:32.871841 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:45:32.876611 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-05-05 22:45:32.876688 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:45:32.905351 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_account_account_id".
2020-05-05 22:45:32.905432 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:45:32.913291 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_account_account_id".
2020-05-05 22:45:32.913370 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:45:32.920850 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:45:32.920925 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:45:32.926816 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:45:32.926894 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:45:32.933061 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_user_id".
2020-05-05 22:45:32.933141 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:45:32.939127 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_user_id".
2020-05-05 22:45:32.939202 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:45:32.945579 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_role_user_role_id".
2020-05-05 22:45:32.945658 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:45:32.951562 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_role_user_role_id".
2020-05-05 22:45:32.951637 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:45:32.984798 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1098f3750>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109a17b50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109a27390>]}
2020-05-05 22:45:32.984956 (MainThread): Flushing usage events
2020-05-05 22:45:33.339448 (MainThread): Connection 'test.salesforce_source.unique_stg_salesforce_user_role_user_role_id' was properly closed.
2020-05-05 22:45:33.339687 (MainThread): Encountered an error:
2020-05-05 22:45:33.339872 (MainThread): Compilation Error
  Invalid sources config given in models/base/src_salesforce.yml @ sources: {'name': 'salesforce', 'loader': 'fivetran', 'loaded_at_field': '_fivetran_synced', 'freshness': {'warn_after': {'count': 24, 'period': 'hour'}, 'error_after': {'count': 48, 'period': 'hour'}}, 'tables': [{'name': 'account', 'loaded_at_field': '_fivetran_synced', 'description': 'Represents an individual account, which is an organization or person involved with your business (such as customers, competitors, and partners).', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation'}, {'name': 'name', 'description': 'Name of the account. Maximum size is 255 characters.'}, {'name': 'industry', 'description': 'An industry associated with this account. Maximum size is 40 characters.'}, {'name': 'number_of_employees', 'description': 'Number of employees working at the company represented by this account. Maximum size is eight digits.'}, {'name': 'annual_revenue', 'description': 'Estimated annual revenue of the account.'}]}, {'name': 'opportunity', 'loaded_at_field': '_fivetran_synced', 'description': 'Represents an opportunity, which is a sale or pending deal.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'amount', 'description': 'Estimated total sale amount. For opportunities with products, the amount is the sum of the related products.'}, {'name': 'probability', 'description': None}, {'name': 'created_date', 'description': 'The system generated time stamp of when the opportunity was created.'}, {'name': 'is_won', 'description': 'Boolean for if the opportunity is won.  Controled by stage_name.'}, {'name': 'is_closed', 'description': 'Boolean for if the opportunity is closed.  Controled by stage_name.'}, {'name': 'forecast_category', 'description': "The category of the state of the opportunity.  Default values are 'pipeline','forecast','bestcase','closed', and 'omited'"}, {'name': 'owner_id', 'description': 'ID of the User who has been assigned to work this opportunity.'}, {'name': 'close_date', 'description': 'Date when the opportunity has been, or is expected to close.'}, {'name': 'stage_name', 'description': 'Current stage of this record. The StageName field controls several other fields on an opportunity.'}]}, {'name': 'user', 'freshness': None, 'description': 'Represents a user in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'name', 'description': 'Concatenation of FirstName and LastName fields.'}, {'name': 'city', 'description': 'The city associated with the user.'}, {'name': 'state', 'description': 'The state associated with the user.'}, {'name': 'manager_id', 'description': 'The Id of the user who manages this user.'}, {'name': 'user_role_id', 'description': 'ID of the user’s user_role'}]}, {'name': 'user_role', 'freshness': None, 'description': 'Represents a user role in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation'}, {'name': 'name', 'description': 'The name assigned the the role'}]}]} - at path ['tables'][1]['columns'][2]['description']: None is not of type 'string'
2020-05-05 22:45:33.343285 (MainThread): jsonschema.exceptions.ValidationError: None is not of type 'string'

Failed validating 'type' in schema['properties']['tables']['items']['properties']['columns']['items']['properties']['description']:
    {'default': '', 'type': 'string'}

On instance['tables'][1]['columns'][2]['description']:
    None

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 499, in get_unparsed_target
    source = UnparsedSourceDefinition.from_dict(data)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/hologram/__init__.py", line 594, in from_dict
    cls.validate(data)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/hologram/__init__.py", line 936, in validate
    raise ValidationError.create_from(error) from error
hologram.ValidationError: None is not of type 'string'

Failed validating 'type' in schema['properties']['tables']['items']['properties']['columns']['items']['properties']['description']:
    {'default': '', 'type': 'string'}

On instance['tables'][1]['columns'][2]['description']:
    None

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/main.py", line 81, in main
    results, succeeded = handle_and_check(args)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/main.py", line 159, in handle_and_check
    task, res = run_from_args(parsed)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/main.py", line 212, in run_from_args
    results = task.run()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 351, in run
    self._runtime_initialize()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 107, in _runtime_initialize
    super()._runtime_initialize()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 75, in _runtime_initialize
    self.load_manifest()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 63, in load_manifest
    self.manifest = get_full_manifest(self.config)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/perf_utils.py", line 23, in get_full_manifest
    return load_manifest(config, internal, set_header)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 646, in load_manifest
    return ManifestLoader.load_all(config, internal_manifest, macro_hook)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 336, in load_all
    loader.load(internal_manifest=internal_manifest)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 208, in load
    self.parse_project(project, macro_manifest, old_results)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 182, in parse_project
    self.parse_with_cache(path, parser, old_results)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 138, in parse_with_cache
    parser.parse_file(block)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 302, in parse_file
    for test_block in parser.parse():
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 394, in parse
    for node in self.get_unparsed_target():
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 502, in get_unparsed_target
    raise CompilationException(msg) from exc
dbt.exceptions.CompilationException: Compilation Error
  Invalid sources config given in models/base/src_salesforce.yml @ sources: {'name': 'salesforce', 'loader': 'fivetran', 'loaded_at_field': '_fivetran_synced', 'freshness': {'warn_after': {'count': 24, 'period': 'hour'}, 'error_after': {'count': 48, 'period': 'hour'}}, 'tables': [{'name': 'account', 'loaded_at_field': '_fivetran_synced', 'description': 'Represents an individual account, which is an organization or person involved with your business (such as customers, competitors, and partners).', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation'}, {'name': 'name', 'description': 'Name of the account. Maximum size is 255 characters.'}, {'name': 'industry', 'description': 'An industry associated with this account. Maximum size is 40 characters.'}, {'name': 'number_of_employees', 'description': 'Number of employees working at the company represented by this account. Maximum size is eight digits.'}, {'name': 'annual_revenue', 'description': 'Estimated annual revenue of the account.'}]}, {'name': 'opportunity', 'loaded_at_field': '_fivetran_synced', 'description': 'Represents an opportunity, which is a sale or pending deal.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'amount', 'description': 'Estimated total sale amount. For opportunities with products, the amount is the sum of the related products.'}, {'name': 'probability', 'description': None}, {'name': 'created_date', 'description': 'The system generated time stamp of when the opportunity was created.'}, {'name': 'is_won', 'description': 'Boolean for if the opportunity is won.  Controled by stage_name.'}, {'name': 'is_closed', 'description': 'Boolean for if the opportunity is closed.  Controled by stage_name.'}, {'name': 'forecast_category', 'description': "The category of the state of the opportunity.  Default values are 'pipeline','forecast','bestcase','closed', and 'omited'"}, {'name': 'owner_id', 'description': 'ID of the User who has been assigned to work this opportunity.'}, {'name': 'close_date', 'description': 'Date when the opportunity has been, or is expected to close.'}, {'name': 'stage_name', 'description': 'Current stage of this record. The StageName field controls several other fields on an opportunity.'}]}, {'name': 'user', 'freshness': None, 'description': 'Represents a user in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'name', 'description': 'Concatenation of FirstName and LastName fields.'}, {'name': 'city', 'description': 'The city associated with the user.'}, {'name': 'state', 'description': 'The state associated with the user.'}, {'name': 'manager_id', 'description': 'The Id of the user who manages this user.'}, {'name': 'user_role_id', 'description': 'ID of the user’s user_role'}]}, {'name': 'user_role', 'freshness': None, 'description': 'Represents a user role in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation'}, {'name': 'name', 'description': 'The name assigned the the role'}]}]} - at path ['tables'][1]['columns'][2]['description']: None is not of type 'string'

2020-05-05 22:46:07.067131 (MainThread): Running with dbt=0.16.0
2020-05-05 22:46:07.200566 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, exclude=None, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='/Users/kristin/.dbt', project_dir=None, record_timing_info=None, rpc_method='run', single_threaded=False, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2020-05-05 22:46:07.201517 (MainThread): Tracking: tracking
2020-05-05 22:46:07.207219 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109d20a50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107cbd790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10a234d10>]}
2020-05-05 22:46:07.223835 (MainThread): Partial parsing not enabled
2020-05-05 22:46:07.225430 (MainThread): Parsing macros/core.sql
2020-05-05 22:46:07.229192 (MainThread): Parsing macros/materializations/helpers.sql
2020-05-05 22:46:07.236169 (MainThread): Parsing macros/materializations/snapshot/snapshot_merge.sql
2020-05-05 22:46:07.237861 (MainThread): Parsing macros/materializations/snapshot/strategies.sql
2020-05-05 22:46:07.253118 (MainThread): Parsing macros/materializations/snapshot/snapshot.sql
2020-05-05 22:46:07.282650 (MainThread): Parsing macros/materializations/seed/seed.sql
2020-05-05 22:46:07.301363 (MainThread): Parsing macros/materializations/incremental/helpers.sql
2020-05-05 22:46:07.303245 (MainThread): Parsing macros/materializations/incremental/incremental.sql
2020-05-05 22:46:07.308849 (MainThread): Parsing macros/materializations/common/merge.sql
2020-05-05 22:46:07.320135 (MainThread): Parsing macros/materializations/table/table.sql
2020-05-05 22:46:07.326214 (MainThread): Parsing macros/materializations/view/view.sql
2020-05-05 22:46:07.331878 (MainThread): Parsing macros/materializations/view/create_or_replace_view.sql
2020-05-05 22:46:07.336490 (MainThread): Parsing macros/etc/get_custom_alias.sql
2020-05-05 22:46:07.337506 (MainThread): Parsing macros/etc/query.sql
2020-05-05 22:46:07.338476 (MainThread): Parsing macros/etc/is_incremental.sql
2020-05-05 22:46:07.339981 (MainThread): Parsing macros/etc/get_relation_comment.sql
2020-05-05 22:46:07.341859 (MainThread): Parsing macros/etc/datetime.sql
2020-05-05 22:46:07.349822 (MainThread): Parsing macros/etc/get_custom_schema.sql
2020-05-05 22:46:07.351573 (MainThread): Parsing macros/etc/get_custom_database.sql
2020-05-05 22:46:07.352506 (MainThread): Parsing macros/adapters/common.sql
2020-05-05 22:46:07.389311 (MainThread): Parsing macros/schema_tests/relationships.sql
2020-05-05 22:46:07.390332 (MainThread): Parsing macros/schema_tests/not_null.sql
2020-05-05 22:46:07.391150 (MainThread): Parsing macros/schema_tests/unique.sql
2020-05-05 22:46:07.392199 (MainThread): Parsing macros/schema_tests/accepted_values.sql
2020-05-05 22:46:07.394242 (MainThread): Parsing macros/etc.sql
2020-05-05 22:46:07.394830 (MainThread): Parsing macros/catalog.sql
2020-05-05 22:46:07.401049 (MainThread): Parsing macros/adapters.sql
2020-05-05 22:46:07.419709 (MainThread): Parsing macros/materializations/seed.sql
2020-05-05 22:46:07.421476 (MainThread): Parsing macros/materializations/view.sql
2020-05-05 22:46:07.422765 (MainThread): Parsing macros/materializations/table.sql
2020-05-05 22:46:07.431615 (MainThread): Parsing macros/materializations/incremental.sql
2020-05-05 22:46:07.442940 (MainThread): Parsing macros/materializations/snapshot.sql
2020-05-05 22:46:07.458589 (MainThread): Partial parsing not enabled
2020-05-05 22:46:07.485544 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-05-05 22:46:07.485627 (MainThread): Opening a new connection, currently in state init
2020-05-05 22:46:07.502058 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-05-05 22:46:07.502134 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:07.506649 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-05-05 22:46:07.506722 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:07.511164 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-05-05 22:46:07.511236 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:07.540789 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_account_account_id".
2020-05-05 22:46:07.540886 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:07.548778 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_account_account_id".
2020-05-05 22:46:07.548865 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:07.556286 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:46:07.556359 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:07.562217 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:46:07.562292 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:07.568039 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_user_id".
2020-05-05 22:46:07.568113 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:07.574021 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_user_id".
2020-05-05 22:46:07.574095 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:07.580349 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_role_user_role_id".
2020-05-05 22:46:07.580424 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:07.586282 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_role_user_role_id".
2020-05-05 22:46:07.586355 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:07.686286 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10a43fd10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10a43ffd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10a43f910>]}
2020-05-05 22:46:07.686526 (MainThread): Flushing usage events
2020-05-05 22:46:08.005934 (MainThread): Connection 'test.salesforce_source.unique_stg_salesforce_user_role_user_role_id' was properly closed.
2020-05-05 22:46:08.006151 (MainThread): Encountered an error:
2020-05-05 22:46:08.006336 (MainThread): Compilation Error in model stg_salesforce_opportunity (models/base/stg_salesforce_opportunity.sql)
  Model 'model.salesforce_source.stg_salesforce_opportunity' (models/base/stg_salesforce_opportunity.sql) depends on source 'salesforce.opportunity' which was not found
2020-05-05 22:46:08.010208 (MainThread): Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/main.py", line 81, in main
    results, succeeded = handle_and_check(args)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/main.py", line 159, in handle_and_check
    task, res = run_from_args(parsed)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/main.py", line 212, in run_from_args
    results = task.run()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 351, in run
    self._runtime_initialize()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 107, in _runtime_initialize
    super()._runtime_initialize()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 75, in _runtime_initialize
    self.load_manifest()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 63, in load_manifest
    self.manifest = get_full_manifest(self.config)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/perf_utils.py", line 23, in get_full_manifest
    return load_manifest(config, internal, set_header)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 646, in load_manifest
    return ManifestLoader.load_all(config, internal_manifest, macro_hook)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 338, in load_all
    manifest = loader.create_manifest()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 323, in create_manifest
    self.process_manifest(manifest)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 301, in process_manifest
    process_sources(manifest, project_name)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 587, in process_sources
    _process_sources_for_node(manifest, current_project, node)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 575, in _process_sources_for_node
    table_name)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/utils.py", line 348, in invalid_source_fail_unless_test
    target_table_name)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/exceptions.py", line 502, in source_target_not_found
    raise_compiler_error(msg, model)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/exceptions.py", line 363, in raise_compiler_error
    raise CompilationException(msg, node)
dbt.exceptions.CompilationException: Compilation Error in model stg_salesforce_opportunity (models/base/stg_salesforce_opportunity.sql)
  Model 'model.salesforce_source.stg_salesforce_opportunity' (models/base/stg_salesforce_opportunity.sql) depends on source 'salesforce.opportunity' which was not found

2020-05-05 22:46:21.329895 (MainThread): Running with dbt=0.16.0
2020-05-05 22:46:21.473422 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, exclude=None, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='/Users/kristin/.dbt', project_dir=None, record_timing_info=None, rpc_method='run', single_threaded=False, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2020-05-05 22:46:21.474213 (MainThread): Tracking: tracking
2020-05-05 22:46:21.479474 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107f6c5d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107f6c450>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107f6cc50>]}
2020-05-05 22:46:21.496281 (MainThread): Partial parsing not enabled
2020-05-05 22:46:21.497972 (MainThread): Parsing macros/core.sql
2020-05-05 22:46:21.501754 (MainThread): Parsing macros/materializations/helpers.sql
2020-05-05 22:46:21.508862 (MainThread): Parsing macros/materializations/snapshot/snapshot_merge.sql
2020-05-05 22:46:21.510466 (MainThread): Parsing macros/materializations/snapshot/strategies.sql
2020-05-05 22:46:21.526171 (MainThread): Parsing macros/materializations/snapshot/snapshot.sql
2020-05-05 22:46:21.556103 (MainThread): Parsing macros/materializations/seed/seed.sql
2020-05-05 22:46:21.574900 (MainThread): Parsing macros/materializations/incremental/helpers.sql
2020-05-05 22:46:21.576599 (MainThread): Parsing macros/materializations/incremental/incremental.sql
2020-05-05 22:46:21.582236 (MainThread): Parsing macros/materializations/common/merge.sql
2020-05-05 22:46:21.593573 (MainThread): Parsing macros/materializations/table/table.sql
2020-05-05 22:46:21.599664 (MainThread): Parsing macros/materializations/view/view.sql
2020-05-05 22:46:21.605395 (MainThread): Parsing macros/materializations/view/create_or_replace_view.sql
2020-05-05 22:46:21.609852 (MainThread): Parsing macros/etc/get_custom_alias.sql
2020-05-05 22:46:21.610716 (MainThread): Parsing macros/etc/query.sql
2020-05-05 22:46:21.611695 (MainThread): Parsing macros/etc/is_incremental.sql
2020-05-05 22:46:21.613223 (MainThread): Parsing macros/etc/get_relation_comment.sql
2020-05-05 22:46:21.615107 (MainThread): Parsing macros/etc/datetime.sql
2020-05-05 22:46:21.623213 (MainThread): Parsing macros/etc/get_custom_schema.sql
2020-05-05 22:46:21.625005 (MainThread): Parsing macros/etc/get_custom_database.sql
2020-05-05 22:46:21.625963 (MainThread): Parsing macros/adapters/common.sql
2020-05-05 22:46:21.663187 (MainThread): Parsing macros/schema_tests/relationships.sql
2020-05-05 22:46:21.664212 (MainThread): Parsing macros/schema_tests/not_null.sql
2020-05-05 22:46:21.665035 (MainThread): Parsing macros/schema_tests/unique.sql
2020-05-05 22:46:21.666006 (MainThread): Parsing macros/schema_tests/accepted_values.sql
2020-05-05 22:46:21.668056 (MainThread): Parsing macros/etc.sql
2020-05-05 22:46:21.668683 (MainThread): Parsing macros/catalog.sql
2020-05-05 22:46:21.677018 (MainThread): Parsing macros/adapters.sql
2020-05-05 22:46:21.700262 (MainThread): Parsing macros/materializations/seed.sql
2020-05-05 22:46:21.702124 (MainThread): Parsing macros/materializations/view.sql
2020-05-05 22:46:21.703450 (MainThread): Parsing macros/materializations/table.sql
2020-05-05 22:46:21.712915 (MainThread): Parsing macros/materializations/incremental.sql
2020-05-05 22:46:21.724281 (MainThread): Parsing macros/materializations/snapshot.sql
2020-05-05 22:46:21.740082 (MainThread): Partial parsing not enabled
2020-05-05 22:46:21.767763 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-05-05 22:46:21.767847 (MainThread): Opening a new connection, currently in state init
2020-05-05 22:46:21.784422 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-05-05 22:46:21.784502 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:21.789133 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-05-05 22:46:21.789209 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:21.793772 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-05-05 22:46:21.793847 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:21.822554 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_account_account_id".
2020-05-05 22:46:21.822631 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:21.830604 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_account_account_id".
2020-05-05 22:46:21.830679 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:21.838147 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:46:21.838224 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:21.844105 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:46:21.844195 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:21.849957 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_user_id".
2020-05-05 22:46:21.850031 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:21.856004 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_user_id".
2020-05-05 22:46:21.856082 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:21.861926 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_role_user_role_id".
2020-05-05 22:46:21.861999 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:21.867771 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_role_user_role_id".
2020-05-05 22:46:21.867856 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:22.135496 (MainThread): Found 4 models, 8 tests, 0 snapshots, 0 analyses, 141 macros, 0 operations, 0 seed files, 4 sources
2020-05-05 22:46:22.142300 (MainThread): 
2020-05-05 22:46:22.142572 (MainThread): Acquiring new bigquery connection "master".
2020-05-05 22:46:22.142650 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:22.711459 (ThreadPoolExecutor-1_0): Acquiring new bigquery connection "list_digital-arbor-400_dbt_kristin_test".
2020-05-05 22:46:22.711807 (ThreadPoolExecutor-1_0): Opening a new connection, currently in state init
2020-05-05 22:46:22.712393 (ThreadPoolExecutor-1_0): Client.dataset is deprecated and will be removed in a future version. Use a string like 'my_project.my_dataset' or a cloud.google.bigquery.DatasetReference object, instead.
2020-05-05 22:46:23.241619 (MainThread): 15:46:23 | Concurrency: 4 threads (target='dev')
2020-05-05 22:46:23.241995 (MainThread): 15:46:23 | 
2020-05-05 22:46:23.245198 (Thread-1): Began running node model.salesforce_source.stg_salesforce_account
2020-05-05 22:46:23.245451 (Thread-2): Began running node model.salesforce_source.stg_salesforce_opportunity
2020-05-05 22:46:23.245826 (Thread-1): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-05-05 22:46:23.245947 (Thread-3): Began running node model.salesforce_source.stg_salesforce_user
2020-05-05 22:46:23.246131 (Thread-4): Began running node model.salesforce_source.stg_salesforce_user_role
2020-05-05 22:46:23.246476 (Thread-2): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-05-05 22:46:23.246620 (Thread-1): Re-using an available connection from the pool (formerly list_digital-arbor-400_dbt_kristin_test).
2020-05-05 22:46:23.246933 (Thread-3): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-05-05 22:46:23.247241 (Thread-4): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-05-05 22:46:23.247373 (Thread-2): Opening a new connection, currently in state init
2020-05-05 22:46:23.247512 (Thread-1): Compiling model.salesforce_source.stg_salesforce_account
2020-05-05 22:46:23.247620 (Thread-3): Opening a new connection, currently in state init
2020-05-05 22:46:23.247722 (Thread-4): Opening a new connection, currently in state init
2020-05-05 22:46:23.247839 (Thread-2): Compiling model.salesforce_source.stg_salesforce_opportunity
2020-05-05 22:46:23.267560 (Thread-3): Compiling model.salesforce_source.stg_salesforce_user
2020-05-05 22:46:23.270465 (Thread-1): Writing injected SQL for node "model.salesforce_source.stg_salesforce_account"
2020-05-05 22:46:23.270729 (Thread-4): Compiling model.salesforce_source.stg_salesforce_user_role
2020-05-05 22:46:23.292643 (Thread-3): Writing injected SQL for node "model.salesforce_source.stg_salesforce_user"
2020-05-05 22:46:23.300251 (Thread-4): Writing injected SQL for node "model.salesforce_source.stg_salesforce_user_role"
2020-05-05 22:46:23.300888 (Thread-3): finished collecting timing info
2020-05-05 22:46:23.301401 (Thread-3): Finished running node model.salesforce_source.stg_salesforce_user
2020-05-05 22:46:23.301853 (Thread-2): finished collecting timing info
2020-05-05 22:46:23.302490 (Thread-4): finished collecting timing info
2020-05-05 22:46:23.302887 (Thread-4): Finished running node model.salesforce_source.stg_salesforce_user_role
2020-05-05 22:46:23.303074 (Thread-1): finished collecting timing info
2020-05-05 22:46:23.303448 (Thread-1): Finished running node model.salesforce_source.stg_salesforce_account
2020-05-05 22:46:23.302224 (Thread-2): Compilation Error in model stg_salesforce_opportunity (models/base/stg_salesforce_opportunity.sql)
  'dbt_utils' is undefined
Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 386, in catch_jinja
    yield
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 409, in render_template
    return template.render(ctx)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "<template>", line 22, in top-level template code
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/sandbox.py", line 407, in getattr
    value = getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'dbt_utils' is undefined

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/node_runners.py", line 223, in safe_run
    result = self.compile_and_execute(manifest, ctx)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/node_runners.py", line 160, in compile_and_execute
    ctx.node = self.compile(manifest)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/node_runners.py", line 335, in compile
    return compile_node(self.adapter, self.config, self.node, manifest, {})
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/compilation.py", line 250, in compile_node
    node = compiler.compile_node(node, manifest, extra_context)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/compilation.py", line 157, in compile_node
    node)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 415, in get_rendered
    return render_template(template, ctx, node)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 409, in render_template
    return template.render(ctx)
  File "/usr/local/opt/python/Frameworks/Python.framework/Versions/3.7/lib/python3.7/contextlib.py", line 130, in __exit__
    self.gen.throw(type, value, traceback)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 391, in catch_jinja
    raise CompilationException(str(e), node) from e
dbt.exceptions.CompilationException: Compilation Error in model stg_salesforce_opportunity (models/base/stg_salesforce_opportunity.sql)
  'dbt_utils' is undefined
2020-05-05 22:46:23.308946 (Thread-2): Finished running node model.salesforce_source.stg_salesforce_opportunity
2020-05-05 22:46:23.347272 (MainThread): 15:46:23 | 
2020-05-05 22:46:23.347559 (MainThread): 15:46:23 | Finished running  in 1.20s.
2020-05-05 22:46:23.347749 (MainThread): Connection 'master' was left open.
2020-05-05 22:46:23.347890 (MainThread): Connection 'model.salesforce_source.stg_salesforce_account' was left open.
2020-05-05 22:46:23.348019 (MainThread): Connection 'model.salesforce_source.stg_salesforce_opportunity' was properly closed.
2020-05-05 22:46:23.348145 (MainThread): Connection 'model.salesforce_source.stg_salesforce_user' was properly closed.
2020-05-05 22:46:23.348268 (MainThread): Connection 'model.salesforce_source.stg_salesforce_user_role' was properly closed.
2020-05-05 22:46:23.349120 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1086d8490>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108634a10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1086d0e50>]}
2020-05-05 22:46:23.349381 (MainThread): Flushing usage events
2020-05-05 22:46:30.298378 (MainThread): Running with dbt=0.16.0
2020-05-05 22:46:30.435119 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, exclude=None, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='/Users/kristin/.dbt', project_dir=None, record_timing_info=None, rpc_method='run', single_threaded=False, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2020-05-05 22:46:30.436025 (MainThread): Tracking: tracking
2020-05-05 22:46:30.441156 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103e2bb50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10390f490>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103e45750>]}
2020-05-05 22:46:30.458189 (MainThread): Partial parsing not enabled
2020-05-05 22:46:30.459888 (MainThread): Parsing macros/core.sql
2020-05-05 22:46:30.463719 (MainThread): Parsing macros/materializations/helpers.sql
2020-05-05 22:46:30.471016 (MainThread): Parsing macros/materializations/snapshot/snapshot_merge.sql
2020-05-05 22:46:30.472652 (MainThread): Parsing macros/materializations/snapshot/strategies.sql
2020-05-05 22:46:30.488646 (MainThread): Parsing macros/materializations/snapshot/snapshot.sql
2020-05-05 22:46:30.519090 (MainThread): Parsing macros/materializations/seed/seed.sql
2020-05-05 22:46:30.538698 (MainThread): Parsing macros/materializations/incremental/helpers.sql
2020-05-05 22:46:30.540466 (MainThread): Parsing macros/materializations/incremental/incremental.sql
2020-05-05 22:46:30.546479 (MainThread): Parsing macros/materializations/common/merge.sql
2020-05-05 22:46:30.557778 (MainThread): Parsing macros/materializations/table/table.sql
2020-05-05 22:46:30.563816 (MainThread): Parsing macros/materializations/view/view.sql
2020-05-05 22:46:30.569396 (MainThread): Parsing macros/materializations/view/create_or_replace_view.sql
2020-05-05 22:46:30.574044 (MainThread): Parsing macros/etc/get_custom_alias.sql
2020-05-05 22:46:30.574903 (MainThread): Parsing macros/etc/query.sql
2020-05-05 22:46:30.575870 (MainThread): Parsing macros/etc/is_incremental.sql
2020-05-05 22:46:30.577366 (MainThread): Parsing macros/etc/get_relation_comment.sql
2020-05-05 22:46:30.579284 (MainThread): Parsing macros/etc/datetime.sql
2020-05-05 22:46:30.587360 (MainThread): Parsing macros/etc/get_custom_schema.sql
2020-05-05 22:46:30.589149 (MainThread): Parsing macros/etc/get_custom_database.sql
2020-05-05 22:46:30.590102 (MainThread): Parsing macros/adapters/common.sql
2020-05-05 22:46:30.627286 (MainThread): Parsing macros/schema_tests/relationships.sql
2020-05-05 22:46:30.628331 (MainThread): Parsing macros/schema_tests/not_null.sql
2020-05-05 22:46:30.629170 (MainThread): Parsing macros/schema_tests/unique.sql
2020-05-05 22:46:30.630157 (MainThread): Parsing macros/schema_tests/accepted_values.sql
2020-05-05 22:46:30.632252 (MainThread): Parsing macros/etc.sql
2020-05-05 22:46:30.632854 (MainThread): Parsing macros/catalog.sql
2020-05-05 22:46:30.639110 (MainThread): Parsing macros/adapters.sql
2020-05-05 22:46:30.658058 (MainThread): Parsing macros/materializations/seed.sql
2020-05-05 22:46:30.659862 (MainThread): Parsing macros/materializations/view.sql
2020-05-05 22:46:30.661175 (MainThread): Parsing macros/materializations/table.sql
2020-05-05 22:46:30.670168 (MainThread): Parsing macros/materializations/incremental.sql
2020-05-05 22:46:30.682001 (MainThread): Parsing macros/materializations/snapshot.sql
2020-05-05 22:46:30.699025 (MainThread): Partial parsing not enabled
2020-05-05 22:46:30.728356 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-05-05 22:46:30.728464 (MainThread): Opening a new connection, currently in state init
2020-05-05 22:46:30.746458 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-05-05 22:46:30.746549 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:30.751731 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-05-05 22:46:30.751807 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:30.756556 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-05-05 22:46:30.756631 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:30.787167 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_account_account_id".
2020-05-05 22:46:30.787271 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:30.797343 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_account_account_id".
2020-05-05 22:46:30.797457 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:30.805880 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:46:30.805974 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:30.812263 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:46:30.812375 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:30.818546 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_user_id".
2020-05-05 22:46:30.818627 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:30.825594 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_user_id".
2020-05-05 22:46:30.825681 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:30.832616 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_role_user_role_id".
2020-05-05 22:46:30.832702 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:30.839775 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_role_user_role_id".
2020-05-05 22:46:30.839867 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:31.105432 (MainThread): Found 4 models, 8 tests, 0 snapshots, 0 analyses, 141 macros, 0 operations, 0 seed files, 4 sources
2020-05-05 22:46:31.112567 (MainThread): 
2020-05-05 22:46:31.112840 (MainThread): Acquiring new bigquery connection "master".
2020-05-05 22:46:31.112962 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:31.646594 (ThreadPoolExecutor-1_0): Acquiring new bigquery connection "list_digital-arbor-400_dbt_kristin_test".
2020-05-05 22:46:31.646935 (ThreadPoolExecutor-1_0): Opening a new connection, currently in state init
2020-05-05 22:46:31.647536 (ThreadPoolExecutor-1_0): Client.dataset is deprecated and will be removed in a future version. Use a string like 'my_project.my_dataset' or a cloud.google.bigquery.DatasetReference object, instead.
2020-05-05 22:46:32.131162 (MainThread): 15:46:32 | Concurrency: 4 threads (target='dev')
2020-05-05 22:46:32.131370 (MainThread): 15:46:32 | 
2020-05-05 22:46:32.133296 (Thread-1): Began running node model.salesforce_source.stg_salesforce_account
2020-05-05 22:46:32.133525 (Thread-2): Began running node model.salesforce_source.stg_salesforce_opportunity
2020-05-05 22:46:32.133906 (Thread-1): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-05-05 22:46:32.134028 (Thread-3): Began running node model.salesforce_source.stg_salesforce_user
2020-05-05 22:46:32.134228 (Thread-4): Began running node model.salesforce_source.stg_salesforce_user_role
2020-05-05 22:46:32.134581 (Thread-2): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-05-05 22:46:32.134723 (Thread-1): Re-using an available connection from the pool (formerly list_digital-arbor-400_dbt_kristin_test).
2020-05-05 22:46:32.135026 (Thread-3): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-05-05 22:46:32.135336 (Thread-4): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-05-05 22:46:32.135471 (Thread-2): Opening a new connection, currently in state init
2020-05-05 22:46:32.135624 (Thread-1): Compiling model.salesforce_source.stg_salesforce_account
2020-05-05 22:46:32.135734 (Thread-3): Opening a new connection, currently in state init
2020-05-05 22:46:32.135874 (Thread-4): Opening a new connection, currently in state init
2020-05-05 22:46:32.136017 (Thread-2): Compiling model.salesforce_source.stg_salesforce_opportunity
2020-05-05 22:46:32.148642 (Thread-3): Compiling model.salesforce_source.stg_salesforce_user
2020-05-05 22:46:32.157483 (Thread-1): Writing injected SQL for node "model.salesforce_source.stg_salesforce_account"
2020-05-05 22:46:32.157686 (Thread-4): Compiling model.salesforce_source.stg_salesforce_user_role
2020-05-05 22:46:32.175533 (Thread-2): finished collecting timing info
2020-05-05 22:46:32.177821 (Thread-3): Writing injected SQL for node "model.salesforce_source.stg_salesforce_user"
2020-05-05 22:46:32.184898 (Thread-4): Writing injected SQL for node "model.salesforce_source.stg_salesforce_user_role"
2020-05-05 22:46:32.186012 (Thread-1): finished collecting timing info
2020-05-05 22:46:32.186418 (Thread-1): Finished running node model.salesforce_source.stg_salesforce_account
2020-05-05 22:46:32.186726 (Thread-3): finished collecting timing info
2020-05-05 22:46:32.187096 (Thread-3): Finished running node model.salesforce_source.stg_salesforce_user
2020-05-05 22:46:32.187671 (Thread-4): finished collecting timing info
2020-05-05 22:46:32.188054 (Thread-4): Finished running node model.salesforce_source.stg_salesforce_user_role
2020-05-05 22:46:32.185251 (Thread-2): Compilation Error in model stg_salesforce_opportunity (models/base/stg_salesforce_opportunity.sql)
  'dbt_utils' is undefined
Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 386, in catch_jinja
    yield
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 409, in render_template
    return template.render(ctx)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "<template>", line 22, in top-level template code
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/sandbox.py", line 407, in getattr
    value = getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'dbt_utils' is undefined

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/node_runners.py", line 223, in safe_run
    result = self.compile_and_execute(manifest, ctx)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/node_runners.py", line 160, in compile_and_execute
    ctx.node = self.compile(manifest)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/node_runners.py", line 335, in compile
    return compile_node(self.adapter, self.config, self.node, manifest, {})
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/compilation.py", line 250, in compile_node
    node = compiler.compile_node(node, manifest, extra_context)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/compilation.py", line 157, in compile_node
    node)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 415, in get_rendered
    return render_template(template, ctx, node)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 409, in render_template
    return template.render(ctx)
  File "/usr/local/opt/python/Frameworks/Python.framework/Versions/3.7/lib/python3.7/contextlib.py", line 130, in __exit__
    self.gen.throw(type, value, traceback)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 391, in catch_jinja
    raise CompilationException(str(e), node) from e
dbt.exceptions.CompilationException: Compilation Error in model stg_salesforce_opportunity (models/base/stg_salesforce_opportunity.sql)
  'dbt_utils' is undefined
2020-05-05 22:46:32.188980 (Thread-2): Finished running node model.salesforce_source.stg_salesforce_opportunity
2020-05-05 22:46:32.236676 (MainThread): 15:46:32 | 
2020-05-05 22:46:32.236964 (MainThread): 15:46:32 | Finished running  in 1.12s.
2020-05-05 22:46:32.237151 (MainThread): Connection 'master' was left open.
2020-05-05 22:46:32.237290 (MainThread): Connection 'model.salesforce_source.stg_salesforce_account' was left open.
2020-05-05 22:46:32.237421 (MainThread): Connection 'model.salesforce_source.stg_salesforce_opportunity' was properly closed.
2020-05-05 22:46:32.237548 (MainThread): Connection 'model.salesforce_source.stg_salesforce_user' was properly closed.
2020-05-05 22:46:32.237670 (MainThread): Connection 'model.salesforce_source.stg_salesforce_user_role' was properly closed.
2020-05-05 22:46:32.238535 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1040c9090>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1040c9d90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104091e50>]}
2020-05-05 22:46:32.238795 (MainThread): Flushing usage events
2020-05-05 22:46:39.787851 (MainThread): Running with dbt=0.16.0
2020-05-05 22:46:39.950169 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, exclude=None, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='/Users/kristin/.dbt', project_dir=None, record_timing_info=None, rpc_method='run', single_threaded=False, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2020-05-05 22:46:39.951143 (MainThread): Tracking: tracking
2020-05-05 22:46:39.957264 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1040eb910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1040d7650>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103bd1710>]}
2020-05-05 22:46:39.977778 (MainThread): Partial parsing not enabled
2020-05-05 22:46:39.979654 (MainThread): Parsing macros/core.sql
2020-05-05 22:46:39.983923 (MainThread): Parsing macros/materializations/helpers.sql
2020-05-05 22:46:39.991403 (MainThread): Parsing macros/materializations/snapshot/snapshot_merge.sql
2020-05-05 22:46:39.993049 (MainThread): Parsing macros/materializations/snapshot/strategies.sql
2020-05-05 22:46:40.009101 (MainThread): Parsing macros/materializations/snapshot/snapshot.sql
2020-05-05 22:46:40.039830 (MainThread): Parsing macros/materializations/seed/seed.sql
2020-05-05 22:46:40.059336 (MainThread): Parsing macros/materializations/incremental/helpers.sql
2020-05-05 22:46:40.061066 (MainThread): Parsing macros/materializations/incremental/incremental.sql
2020-05-05 22:46:40.066829 (MainThread): Parsing macros/materializations/common/merge.sql
2020-05-05 22:46:40.078381 (MainThread): Parsing macros/materializations/table/table.sql
2020-05-05 22:46:40.084604 (MainThread): Parsing macros/materializations/view/view.sql
2020-05-05 22:46:40.090344 (MainThread): Parsing macros/materializations/view/create_or_replace_view.sql
2020-05-05 22:46:40.094980 (MainThread): Parsing macros/etc/get_custom_alias.sql
2020-05-05 22:46:40.095858 (MainThread): Parsing macros/etc/query.sql
2020-05-05 22:46:40.096853 (MainThread): Parsing macros/etc/is_incremental.sql
2020-05-05 22:46:40.098382 (MainThread): Parsing macros/etc/get_relation_comment.sql
2020-05-05 22:46:40.100328 (MainThread): Parsing macros/etc/datetime.sql
2020-05-05 22:46:40.108626 (MainThread): Parsing macros/etc/get_custom_schema.sql
2020-05-05 22:46:40.110449 (MainThread): Parsing macros/etc/get_custom_database.sql
2020-05-05 22:46:40.111427 (MainThread): Parsing macros/adapters/common.sql
2020-05-05 22:46:40.149493 (MainThread): Parsing macros/schema_tests/relationships.sql
2020-05-05 22:46:40.150515 (MainThread): Parsing macros/schema_tests/not_null.sql
2020-05-05 22:46:40.151336 (MainThread): Parsing macros/schema_tests/unique.sql
2020-05-05 22:46:40.152309 (MainThread): Parsing macros/schema_tests/accepted_values.sql
2020-05-05 22:46:40.154365 (MainThread): Parsing macros/etc.sql
2020-05-05 22:46:40.154959 (MainThread): Parsing macros/catalog.sql
2020-05-05 22:46:40.161204 (MainThread): Parsing macros/adapters.sql
2020-05-05 22:46:40.180792 (MainThread): Parsing macros/materializations/seed.sql
2020-05-05 22:46:40.182562 (MainThread): Parsing macros/materializations/view.sql
2020-05-05 22:46:40.183862 (MainThread): Parsing macros/materializations/table.sql
2020-05-05 22:46:40.192761 (MainThread): Parsing macros/materializations/incremental.sql
2020-05-05 22:46:40.204167 (MainThread): Parsing macros/materializations/snapshot.sql
2020-05-05 22:46:40.219818 (MainThread): Partial parsing not enabled
2020-05-05 22:46:40.247022 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-05-05 22:46:40.247103 (MainThread): Opening a new connection, currently in state init
2020-05-05 22:46:40.263874 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-05-05 22:46:40.263956 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:40.271299 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-05-05 22:46:40.271440 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:40.277216 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-05-05 22:46:40.277300 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:40.310918 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_account_account_id".
2020-05-05 22:46:40.311023 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:40.319038 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_account_account_id".
2020-05-05 22:46:40.319117 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:40.326480 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:46:40.326556 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:40.332389 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:46:40.332465 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:40.338258 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_user_id".
2020-05-05 22:46:40.338334 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:40.344226 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_user_id".
2020-05-05 22:46:40.344301 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:40.350441 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_role_user_role_id".
2020-05-05 22:46:40.350520 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:40.357015 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_role_user_role_id".
2020-05-05 22:46:40.357115 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:40.620001 (MainThread): Found 4 models, 8 tests, 0 snapshots, 0 analyses, 141 macros, 0 operations, 0 seed files, 4 sources
2020-05-05 22:46:40.628731 (MainThread): 
2020-05-05 22:46:40.629256 (MainThread): Acquiring new bigquery connection "master".
2020-05-05 22:46:40.629466 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:41.191634 (ThreadPoolExecutor-1_0): Acquiring new bigquery connection "list_digital-arbor-400_dbt_kristin_test".
2020-05-05 22:46:41.191979 (ThreadPoolExecutor-1_0): Opening a new connection, currently in state init
2020-05-05 22:46:41.192583 (ThreadPoolExecutor-1_0): Client.dataset is deprecated and will be removed in a future version. Use a string like 'my_project.my_dataset' or a cloud.google.bigquery.DatasetReference object, instead.
2020-05-05 22:46:41.656988 (MainThread): 15:46:41 | Concurrency: 4 threads (target='dev')
2020-05-05 22:46:41.657194 (MainThread): 15:46:41 | 
2020-05-05 22:46:41.658890 (Thread-1): Began running node model.salesforce_source.stg_salesforce_account
2020-05-05 22:46:41.659083 (Thread-2): Began running node model.salesforce_source.stg_salesforce_opportunity
2020-05-05 22:46:41.659278 (Thread-3): Began running node model.salesforce_source.stg_salesforce_user
2020-05-05 22:46:41.659599 (Thread-1): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-05-05 22:46:41.659716 (Thread-4): Began running node model.salesforce_source.stg_salesforce_user_role
2020-05-05 22:46:41.660047 (Thread-2): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-05-05 22:46:41.660352 (Thread-3): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-05-05 22:46:41.660480 (Thread-1): Re-using an available connection from the pool (formerly list_digital-arbor-400_dbt_kristin_test).
2020-05-05 22:46:41.660760 (Thread-4): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-05-05 22:46:41.660890 (Thread-2): Opening a new connection, currently in state init
2020-05-05 22:46:41.660999 (Thread-3): Opening a new connection, currently in state init
2020-05-05 22:46:41.661162 (Thread-1): Compiling model.salesforce_source.stg_salesforce_account
2020-05-05 22:46:41.661295 (Thread-4): Opening a new connection, currently in state init
2020-05-05 22:46:41.661420 (Thread-2): Compiling model.salesforce_source.stg_salesforce_opportunity
2020-05-05 22:46:41.661534 (Thread-3): Compiling model.salesforce_source.stg_salesforce_user
2020-05-05 22:46:41.680501 (Thread-4): Compiling model.salesforce_source.stg_salesforce_user_role
2020-05-05 22:46:41.683384 (Thread-1): Writing injected SQL for node "model.salesforce_source.stg_salesforce_account"
2020-05-05 22:46:41.703662 (Thread-3): Writing injected SQL for node "model.salesforce_source.stg_salesforce_user"
2020-05-05 22:46:41.710603 (Thread-4): Writing injected SQL for node "model.salesforce_source.stg_salesforce_user_role"
2020-05-05 22:46:41.710930 (Thread-2): finished collecting timing info
2020-05-05 22:46:41.711549 (Thread-1): finished collecting timing info
2020-05-05 22:46:41.712394 (Thread-1): Finished running node model.salesforce_source.stg_salesforce_account
2020-05-05 22:46:41.712902 (Thread-4): finished collecting timing info
2020-05-05 22:46:41.713277 (Thread-4): Finished running node model.salesforce_source.stg_salesforce_user_role
2020-05-05 22:46:41.713473 (Thread-3): finished collecting timing info
2020-05-05 22:46:41.713850 (Thread-3): Finished running node model.salesforce_source.stg_salesforce_user
2020-05-05 22:46:41.711929 (Thread-2): Compilation Error in model stg_salesforce_opportunity (models/base/stg_salesforce_opportunity.sql)
  'dbt_utils' is undefined
Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 386, in catch_jinja
    yield
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 409, in render_template
    return template.render(ctx)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "<template>", line 22, in top-level template code
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/sandbox.py", line 407, in getattr
    value = getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'dbt_utils' is undefined

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/node_runners.py", line 223, in safe_run
    result = self.compile_and_execute(manifest, ctx)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/node_runners.py", line 160, in compile_and_execute
    ctx.node = self.compile(manifest)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/node_runners.py", line 335, in compile
    return compile_node(self.adapter, self.config, self.node, manifest, {})
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/compilation.py", line 250, in compile_node
    node = compiler.compile_node(node, manifest, extra_context)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/compilation.py", line 157, in compile_node
    node)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 415, in get_rendered
    return render_template(template, ctx, node)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 409, in render_template
    return template.render(ctx)
  File "/usr/local/opt/python/Frameworks/Python.framework/Versions/3.7/lib/python3.7/contextlib.py", line 130, in __exit__
    self.gen.throw(type, value, traceback)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 391, in catch_jinja
    raise CompilationException(str(e), node) from e
dbt.exceptions.CompilationException: Compilation Error in model stg_salesforce_opportunity (models/base/stg_salesforce_opportunity.sql)
  'dbt_utils' is undefined
2020-05-05 22:46:41.715116 (Thread-2): Finished running node model.salesforce_source.stg_salesforce_opportunity
2020-05-05 22:46:41.765233 (MainThread): 15:46:41 | 
2020-05-05 22:46:41.765526 (MainThread): 15:46:41 | Finished running  in 1.14s.
2020-05-05 22:46:41.765722 (MainThread): Connection 'master' was left open.
2020-05-05 22:46:41.765867 (MainThread): Connection 'model.salesforce_source.stg_salesforce_account' was left open.
2020-05-05 22:46:41.766004 (MainThread): Connection 'model.salesforce_source.stg_salesforce_opportunity' was properly closed.
2020-05-05 22:46:41.766137 (MainThread): Connection 'model.salesforce_source.stg_salesforce_user' was properly closed.
2020-05-05 22:46:41.766267 (MainThread): Connection 'model.salesforce_source.stg_salesforce_user_role' was properly closed.
2020-05-05 22:46:41.767179 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104255590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104251210>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104255450>]}
2020-05-05 22:46:41.767522 (MainThread): Flushing usage events
2020-05-05 22:46:49.781650 (MainThread): Running with dbt=0.16.0
2020-05-05 22:46:49.913323 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, exclude=None, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='/Users/kristin/.dbt', project_dir=None, record_timing_info=None, rpc_method='run', single_threaded=False, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2020-05-05 22:46:49.914272 (MainThread): Tracking: tracking
2020-05-05 22:46:49.919503 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10a5f2a50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10a5f24d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10a5cfc10>]}
2020-05-05 22:46:49.936909 (MainThread): Partial parsing not enabled
2020-05-05 22:46:49.938580 (MainThread): Parsing macros/core.sql
2020-05-05 22:46:49.942340 (MainThread): Parsing macros/materializations/helpers.sql
2020-05-05 22:46:49.949419 (MainThread): Parsing macros/materializations/snapshot/snapshot_merge.sql
2020-05-05 22:46:49.950987 (MainThread): Parsing macros/materializations/snapshot/strategies.sql
2020-05-05 22:46:49.966306 (MainThread): Parsing macros/materializations/snapshot/snapshot.sql
2020-05-05 22:46:49.995646 (MainThread): Parsing macros/materializations/seed/seed.sql
2020-05-05 22:46:50.014401 (MainThread): Parsing macros/materializations/incremental/helpers.sql
2020-05-05 22:46:50.016092 (MainThread): Parsing macros/materializations/incremental/incremental.sql
2020-05-05 22:46:50.021711 (MainThread): Parsing macros/materializations/common/merge.sql
2020-05-05 22:46:50.033070 (MainThread): Parsing macros/materializations/table/table.sql
2020-05-05 22:46:50.039158 (MainThread): Parsing macros/materializations/view/view.sql
2020-05-05 22:46:50.044779 (MainThread): Parsing macros/materializations/view/create_or_replace_view.sql
2020-05-05 22:46:50.049132 (MainThread): Parsing macros/etc/get_custom_alias.sql
2020-05-05 22:46:50.049974 (MainThread): Parsing macros/etc/query.sql
2020-05-05 22:46:50.050929 (MainThread): Parsing macros/etc/is_incremental.sql
2020-05-05 22:46:50.052406 (MainThread): Parsing macros/etc/get_relation_comment.sql
2020-05-05 22:46:50.054259 (MainThread): Parsing macros/etc/datetime.sql
2020-05-05 22:46:50.062494 (MainThread): Parsing macros/etc/get_custom_schema.sql
2020-05-05 22:46:50.064249 (MainThread): Parsing macros/etc/get_custom_database.sql
2020-05-05 22:46:50.065187 (MainThread): Parsing macros/adapters/common.sql
2020-05-05 22:46:50.103593 (MainThread): Parsing macros/schema_tests/relationships.sql
2020-05-05 22:46:50.104681 (MainThread): Parsing macros/schema_tests/not_null.sql
2020-05-05 22:46:50.105508 (MainThread): Parsing macros/schema_tests/unique.sql
2020-05-05 22:46:50.106481 (MainThread): Parsing macros/schema_tests/accepted_values.sql
2020-05-05 22:46:50.108542 (MainThread): Parsing macros/etc.sql
2020-05-05 22:46:50.109251 (MainThread): Parsing macros/catalog.sql
2020-05-05 22:46:50.115543 (MainThread): Parsing macros/adapters.sql
2020-05-05 22:46:50.134486 (MainThread): Parsing macros/materializations/seed.sql
2020-05-05 22:46:50.136539 (MainThread): Parsing macros/materializations/view.sql
2020-05-05 22:46:50.137862 (MainThread): Parsing macros/materializations/table.sql
2020-05-05 22:46:50.146788 (MainThread): Parsing macros/materializations/incremental.sql
2020-05-05 22:46:50.158188 (MainThread): Parsing macros/materializations/snapshot.sql
2020-05-05 22:46:50.173980 (MainThread): Partial parsing not enabled
2020-05-05 22:46:50.201616 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-05-05 22:46:50.201701 (MainThread): Opening a new connection, currently in state init
2020-05-05 22:46:50.218194 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-05-05 22:46:50.218276 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:50.222921 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-05-05 22:46:50.222998 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:50.227517 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-05-05 22:46:50.227594 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:50.256146 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_account_account_id".
2020-05-05 22:46:50.256225 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:50.264099 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_account_account_id".
2020-05-05 22:46:50.264177 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:50.271846 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:46:50.271923 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:50.277852 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:46:50.277930 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:50.283785 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_user_id".
2020-05-05 22:46:50.283863 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:50.289812 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_user_id".
2020-05-05 22:46:50.289889 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:50.295803 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_role_user_role_id".
2020-05-05 22:46:50.295880 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:50.301943 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_role_user_role_id".
2020-05-05 22:46:50.302020 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:46:50.332278 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ab7efd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10acfb350>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ad68e90>]}
2020-05-05 22:46:50.332444 (MainThread): Flushing usage events
2020-05-05 22:46:50.684208 (MainThread): Connection 'test.salesforce_source.unique_stg_salesforce_user_role_user_role_id' was properly closed.
2020-05-05 22:46:50.684461 (MainThread): Encountered an error:
2020-05-05 22:46:50.684648 (MainThread): Compilation Error
  Invalid sources config given in models/base/src_salesforce.yml @ sources: {'name': 'salesforce', 'loader': 'fivetran', 'loaded_at_field': '_fivetran_synced', 'freshness': {'warn_after': {'count': 24, 'period': 'hour'}, 'error_after': {'count': 48, 'period': 'hour'}}, 'tables': [{'name': 'account'}, {'name': 'opportunity', 'loaded_at_field': '_fivetran_synced', 'description': 'Represents an opportunity, which is a sale or pending deal.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'amount', 'description': 'Estimated total sale amount. For opportunities with products, the amount is the sum of the related products.'}, {'name': 'probability', 'description': None}, {'name': 'created_date', 'description': 'The system generated time stamp of when the opportunity was created.'}, {'name': 'is_won', 'description': 'Boolean for if the opportunity is won.  Controled by stage_name.'}, {'name': 'is_closed', 'description': 'Boolean for if the opportunity is closed.  Controled by stage_name.'}, {'name': 'forecast_category', 'description': "The category of the state of the opportunity.  Default values are 'pipeline','forecast','bestcase','closed', and 'omited'"}, {'name': 'owner_id', 'description': 'ID of the User who has been assigned to work this opportunity.'}, {'name': 'close_date', 'description': 'Date when the opportunity has been, or is expected to close.'}, {'name': 'stage_name', 'description': 'Current stage of this record. The StageName field controls several other fields on an opportunity.'}]}, {'name': 'user', 'freshness': None, 'description': 'Represents a user in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'name', 'description': 'Concatenation of FirstName and LastName fields.'}, {'name': 'city', 'description': 'The city associated with the user.'}, {'name': 'state', 'description': 'The state associated with the user.'}, {'name': 'manager_id', 'description': 'The Id of the user who manages this user.'}, {'name': 'user_role_id', 'description': 'ID of the user’s user_role'}]}, {'name': 'user_role', 'freshness': None, 'description': 'Represents a user role in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation'}, {'name': 'name', 'description': 'The name assigned the the role'}]}]} - at path ['tables'][1]['columns'][2]['description']: None is not of type 'string'
2020-05-05 22:46:50.687243 (MainThread): jsonschema.exceptions.ValidationError: None is not of type 'string'

Failed validating 'type' in schema['properties']['tables']['items']['properties']['columns']['items']['properties']['description']:
    {'default': '', 'type': 'string'}

On instance['tables'][1]['columns'][2]['description']:
    None

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 499, in get_unparsed_target
    source = UnparsedSourceDefinition.from_dict(data)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/hologram/__init__.py", line 594, in from_dict
    cls.validate(data)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/hologram/__init__.py", line 936, in validate
    raise ValidationError.create_from(error) from error
hologram.ValidationError: None is not of type 'string'

Failed validating 'type' in schema['properties']['tables']['items']['properties']['columns']['items']['properties']['description']:
    {'default': '', 'type': 'string'}

On instance['tables'][1]['columns'][2]['description']:
    None

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/main.py", line 81, in main
    results, succeeded = handle_and_check(args)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/main.py", line 159, in handle_and_check
    task, res = run_from_args(parsed)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/main.py", line 212, in run_from_args
    results = task.run()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 351, in run
    self._runtime_initialize()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 107, in _runtime_initialize
    super()._runtime_initialize()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 75, in _runtime_initialize
    self.load_manifest()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 63, in load_manifest
    self.manifest = get_full_manifest(self.config)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/perf_utils.py", line 23, in get_full_manifest
    return load_manifest(config, internal, set_header)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 646, in load_manifest
    return ManifestLoader.load_all(config, internal_manifest, macro_hook)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 336, in load_all
    loader.load(internal_manifest=internal_manifest)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 208, in load
    self.parse_project(project, macro_manifest, old_results)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 182, in parse_project
    self.parse_with_cache(path, parser, old_results)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 138, in parse_with_cache
    parser.parse_file(block)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 302, in parse_file
    for test_block in parser.parse():
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 394, in parse
    for node in self.get_unparsed_target():
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 502, in get_unparsed_target
    raise CompilationException(msg) from exc
dbt.exceptions.CompilationException: Compilation Error
  Invalid sources config given in models/base/src_salesforce.yml @ sources: {'name': 'salesforce', 'loader': 'fivetran', 'loaded_at_field': '_fivetran_synced', 'freshness': {'warn_after': {'count': 24, 'period': 'hour'}, 'error_after': {'count': 48, 'period': 'hour'}}, 'tables': [{'name': 'account'}, {'name': 'opportunity', 'loaded_at_field': '_fivetran_synced', 'description': 'Represents an opportunity, which is a sale or pending deal.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'amount', 'description': 'Estimated total sale amount. For opportunities with products, the amount is the sum of the related products.'}, {'name': 'probability', 'description': None}, {'name': 'created_date', 'description': 'The system generated time stamp of when the opportunity was created.'}, {'name': 'is_won', 'description': 'Boolean for if the opportunity is won.  Controled by stage_name.'}, {'name': 'is_closed', 'description': 'Boolean for if the opportunity is closed.  Controled by stage_name.'}, {'name': 'forecast_category', 'description': "The category of the state of the opportunity.  Default values are 'pipeline','forecast','bestcase','closed', and 'omited'"}, {'name': 'owner_id', 'description': 'ID of the User who has been assigned to work this opportunity.'}, {'name': 'close_date', 'description': 'Date when the opportunity has been, or is expected to close.'}, {'name': 'stage_name', 'description': 'Current stage of this record. The StageName field controls several other fields on an opportunity.'}]}, {'name': 'user', 'freshness': None, 'description': 'Represents a user in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'name', 'description': 'Concatenation of FirstName and LastName fields.'}, {'name': 'city', 'description': 'The city associated with the user.'}, {'name': 'state', 'description': 'The state associated with the user.'}, {'name': 'manager_id', 'description': 'The Id of the user who manages this user.'}, {'name': 'user_role_id', 'description': 'ID of the user’s user_role'}]}, {'name': 'user_role', 'freshness': None, 'description': 'Represents a user role in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation'}, {'name': 'name', 'description': 'The name assigned the the role'}]}]} - at path ['tables'][1]['columns'][2]['description']: None is not of type 'string'

2020-05-05 22:47:03.324695 (MainThread): Running with dbt=0.16.0
2020-05-05 22:47:03.463269 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, exclude=None, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='/Users/kristin/.dbt', project_dir=None, record_timing_info=None, rpc_method='run', single_threaded=False, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2020-05-05 22:47:03.464332 (MainThread): Tracking: tracking
2020-05-05 22:47:03.470177 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10bfae210>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10c4aa390>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10c4aa990>]}
2020-05-05 22:47:03.487357 (MainThread): Partial parsing not enabled
2020-05-05 22:47:03.489018 (MainThread): Parsing macros/core.sql
2020-05-05 22:47:03.492998 (MainThread): Parsing macros/materializations/helpers.sql
2020-05-05 22:47:03.500478 (MainThread): Parsing macros/materializations/snapshot/snapshot_merge.sql
2020-05-05 22:47:03.502116 (MainThread): Parsing macros/materializations/snapshot/strategies.sql
2020-05-05 22:47:03.518293 (MainThread): Parsing macros/materializations/snapshot/snapshot.sql
2020-05-05 22:47:03.549004 (MainThread): Parsing macros/materializations/seed/seed.sql
2020-05-05 22:47:03.568577 (MainThread): Parsing macros/materializations/incremental/helpers.sql
2020-05-05 22:47:03.570324 (MainThread): Parsing macros/materializations/incremental/incremental.sql
2020-05-05 22:47:03.576279 (MainThread): Parsing macros/materializations/common/merge.sql
2020-05-05 22:47:03.587961 (MainThread): Parsing macros/materializations/table/table.sql
2020-05-05 22:47:03.594245 (MainThread): Parsing macros/materializations/view/view.sql
2020-05-05 22:47:03.600071 (MainThread): Parsing macros/materializations/view/create_or_replace_view.sql
2020-05-05 22:47:03.604634 (MainThread): Parsing macros/etc/get_custom_alias.sql
2020-05-05 22:47:03.605587 (MainThread): Parsing macros/etc/query.sql
2020-05-05 22:47:03.606588 (MainThread): Parsing macros/etc/is_incremental.sql
2020-05-05 22:47:03.608129 (MainThread): Parsing macros/etc/get_relation_comment.sql
2020-05-05 22:47:03.610094 (MainThread): Parsing macros/etc/datetime.sql
2020-05-05 22:47:03.618482 (MainThread): Parsing macros/etc/get_custom_schema.sql
2020-05-05 22:47:03.620319 (MainThread): Parsing macros/etc/get_custom_database.sql
2020-05-05 22:47:03.621305 (MainThread): Parsing macros/adapters/common.sql
2020-05-05 22:47:03.659714 (MainThread): Parsing macros/schema_tests/relationships.sql
2020-05-05 22:47:03.660742 (MainThread): Parsing macros/schema_tests/not_null.sql
2020-05-05 22:47:03.661564 (MainThread): Parsing macros/schema_tests/unique.sql
2020-05-05 22:47:03.662695 (MainThread): Parsing macros/schema_tests/accepted_values.sql
2020-05-05 22:47:03.664755 (MainThread): Parsing macros/etc.sql
2020-05-05 22:47:03.665462 (MainThread): Parsing macros/catalog.sql
2020-05-05 22:47:03.671980 (MainThread): Parsing macros/adapters.sql
2020-05-05 22:47:03.691018 (MainThread): Parsing macros/materializations/seed.sql
2020-05-05 22:47:03.692787 (MainThread): Parsing macros/materializations/view.sql
2020-05-05 22:47:03.694085 (MainThread): Parsing macros/materializations/table.sql
2020-05-05 22:47:03.703310 (MainThread): Parsing macros/materializations/incremental.sql
2020-05-05 22:47:03.714919 (MainThread): Parsing macros/materializations/snapshot.sql
2020-05-05 22:47:03.730895 (MainThread): Partial parsing not enabled
2020-05-05 22:47:03.760309 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-05-05 22:47:03.760478 (MainThread): Opening a new connection, currently in state init
2020-05-05 22:47:03.779288 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-05-05 22:47:03.779403 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:47:03.784203 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-05-05 22:47:03.784285 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:47:03.789006 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-05-05 22:47:03.789084 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:47:03.818820 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_account_account_id".
2020-05-05 22:47:03.818915 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:47:03.827096 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_account_account_id".
2020-05-05 22:47:03.827176 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:47:03.834941 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:47:03.835021 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:47:03.841224 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:47:03.841305 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:47:03.847196 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_user_id".
2020-05-05 22:47:03.847275 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:47:03.853451 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_user_id".
2020-05-05 22:47:03.853531 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:47:03.859555 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_role_user_role_id".
2020-05-05 22:47:03.859633 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:47:03.865846 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_role_user_role_id".
2020-05-05 22:47:03.865929 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:47:03.896458 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10c4b9190>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10c6b80d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10c6d7150>]}
2020-05-05 22:47:03.896627 (MainThread): Flushing usage events
2020-05-05 22:47:04.233306 (MainThread): Connection 'test.salesforce_source.unique_stg_salesforce_user_role_user_role_id' was properly closed.
2020-05-05 22:47:04.233539 (MainThread): Encountered an error:
2020-05-05 22:47:04.233735 (MainThread): Compilation Error
  Invalid sources config given in models/base/src_salesforce.yml @ sources: {'name': 'salesforce', 'loader': 'fivetran', 'loaded_at_field': '_fivetran_synced', 'freshness': {'warn_after': {'count': 24, 'period': 'hour'}, 'error_after': {'count': 48, 'period': 'hour'}}, 'tables': [{'name': 'account'}, {'name': 'opportunity', 'description': 'Represents an opportunity, which is a sale or pending deal.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'amount', 'description': 'Estimated total sale amount. For opportunities with products, the amount is the sum of the related products.'}, {'name': 'probability', 'description': None}, {'name': 'created_date', 'description': 'The system generated time stamp of when the opportunity was created.'}, {'name': 'is_won', 'description': 'Boolean for if the opportunity is won.  Controled by stage_name.'}, {'name': 'is_closed', 'description': 'Boolean for if the opportunity is closed.  Controled by stage_name.'}, {'name': 'forecast_category', 'description': "The category of the state of the opportunity.  Default values are 'pipeline','forecast','bestcase','closed', and 'omited'"}, {'name': 'owner_id', 'description': 'ID of the User who has been assigned to work this opportunity.'}, {'name': 'close_date', 'description': 'Date when the opportunity has been, or is expected to close.'}, {'name': 'stage_name', 'description': 'Current stage of this record. The StageName field controls several other fields on an opportunity.'}]}, {'name': 'user', 'freshness': None, 'description': 'Represents a user in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'name', 'description': 'Concatenation of FirstName and LastName fields.'}, {'name': 'city', 'description': 'The city associated with the user.'}, {'name': 'state', 'description': 'The state associated with the user.'}, {'name': 'manager_id', 'description': 'The Id of the user who manages this user.'}, {'name': 'user_role_id', 'description': 'ID of the user’s user_role'}]}, {'name': 'user_role', 'freshness': None, 'description': 'Represents a user role in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation'}, {'name': 'name', 'description': 'The name assigned the the role'}]}]} - at path ['tables'][1]['columns'][2]['description']: None is not of type 'string'
2020-05-05 22:47:04.236315 (MainThread): jsonschema.exceptions.ValidationError: None is not of type 'string'

Failed validating 'type' in schema['properties']['tables']['items']['properties']['columns']['items']['properties']['description']:
    {'default': '', 'type': 'string'}

On instance['tables'][1]['columns'][2]['description']:
    None

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 499, in get_unparsed_target
    source = UnparsedSourceDefinition.from_dict(data)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/hologram/__init__.py", line 594, in from_dict
    cls.validate(data)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/hologram/__init__.py", line 936, in validate
    raise ValidationError.create_from(error) from error
hologram.ValidationError: None is not of type 'string'

Failed validating 'type' in schema['properties']['tables']['items']['properties']['columns']['items']['properties']['description']:
    {'default': '', 'type': 'string'}

On instance['tables'][1]['columns'][2]['description']:
    None

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/main.py", line 81, in main
    results, succeeded = handle_and_check(args)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/main.py", line 159, in handle_and_check
    task, res = run_from_args(parsed)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/main.py", line 212, in run_from_args
    results = task.run()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 351, in run
    self._runtime_initialize()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 107, in _runtime_initialize
    super()._runtime_initialize()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 75, in _runtime_initialize
    self.load_manifest()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/task/runnable.py", line 63, in load_manifest
    self.manifest = get_full_manifest(self.config)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/perf_utils.py", line 23, in get_full_manifest
    return load_manifest(config, internal, set_header)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 646, in load_manifest
    return ManifestLoader.load_all(config, internal_manifest, macro_hook)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 336, in load_all
    loader.load(internal_manifest=internal_manifest)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 208, in load
    self.parse_project(project, macro_manifest, old_results)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 182, in parse_project
    self.parse_with_cache(path, parser, old_results)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/manifest.py", line 138, in parse_with_cache
    parser.parse_file(block)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 302, in parse_file
    for test_block in parser.parse():
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 394, in parse
    for node in self.get_unparsed_target():
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/parser/schemas.py", line 502, in get_unparsed_target
    raise CompilationException(msg) from exc
dbt.exceptions.CompilationException: Compilation Error
  Invalid sources config given in models/base/src_salesforce.yml @ sources: {'name': 'salesforce', 'loader': 'fivetran', 'loaded_at_field': '_fivetran_synced', 'freshness': {'warn_after': {'count': 24, 'period': 'hour'}, 'error_after': {'count': 48, 'period': 'hour'}}, 'tables': [{'name': 'account'}, {'name': 'opportunity', 'description': 'Represents an opportunity, which is a sale or pending deal.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'amount', 'description': 'Estimated total sale amount. For opportunities with products, the amount is the sum of the related products.'}, {'name': 'probability', 'description': None}, {'name': 'created_date', 'description': 'The system generated time stamp of when the opportunity was created.'}, {'name': 'is_won', 'description': 'Boolean for if the opportunity is won.  Controled by stage_name.'}, {'name': 'is_closed', 'description': 'Boolean for if the opportunity is closed.  Controled by stage_name.'}, {'name': 'forecast_category', 'description': "The category of the state of the opportunity.  Default values are 'pipeline','forecast','bestcase','closed', and 'omited'"}, {'name': 'owner_id', 'description': 'ID of the User who has been assigned to work this opportunity.'}, {'name': 'close_date', 'description': 'Date when the opportunity has been, or is expected to close.'}, {'name': 'stage_name', 'description': 'Current stage of this record. The StageName field controls several other fields on an opportunity.'}]}, {'name': 'user', 'freshness': None, 'description': 'Represents a user in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation.'}, {'name': 'name', 'description': 'Concatenation of FirstName and LastName fields.'}, {'name': 'city', 'description': 'The city associated with the user.'}, {'name': 'state', 'description': 'The state associated with the user.'}, {'name': 'manager_id', 'description': 'The Id of the user who manages this user.'}, {'name': 'user_role_id', 'description': 'ID of the user’s user_role'}]}, {'name': 'user_role', 'freshness': None, 'description': 'Represents a user role in your organization.', 'columns': [{'name': 'id', 'description': 'the unique, system-generated ID assigned during creation'}, {'name': 'name', 'description': 'The name assigned the the role'}]}]} - at path ['tables'][1]['columns'][2]['description']: None is not of type 'string'

2020-05-05 22:49:11.310377 (MainThread): Running with dbt=0.16.0
2020-05-05 22:49:11.442584 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, exclude=None, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='/Users/kristin/.dbt', project_dir=None, record_timing_info=None, rpc_method='run', single_threaded=False, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2020-05-05 22:49:11.443515 (MainThread): Tracking: tracking
2020-05-05 22:49:11.448900 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1052dcdd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1052f1390>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1052dc510>]}
2020-05-05 22:49:11.465565 (MainThread): Partial parsing not enabled
2020-05-05 22:49:11.467262 (MainThread): Parsing macros/core.sql
2020-05-05 22:49:11.470994 (MainThread): Parsing macros/materializations/helpers.sql
2020-05-05 22:49:11.478368 (MainThread): Parsing macros/materializations/snapshot/snapshot_merge.sql
2020-05-05 22:49:11.479961 (MainThread): Parsing macros/materializations/snapshot/strategies.sql
2020-05-05 22:49:11.495467 (MainThread): Parsing macros/materializations/snapshot/snapshot.sql
2020-05-05 22:49:11.525335 (MainThread): Parsing macros/materializations/seed/seed.sql
2020-05-05 22:49:11.544326 (MainThread): Parsing macros/materializations/incremental/helpers.sql
2020-05-05 22:49:11.546045 (MainThread): Parsing macros/materializations/incremental/incremental.sql
2020-05-05 22:49:11.551699 (MainThread): Parsing macros/materializations/common/merge.sql
2020-05-05 22:49:11.562832 (MainThread): Parsing macros/materializations/table/table.sql
2020-05-05 22:49:11.568992 (MainThread): Parsing macros/materializations/view/view.sql
2020-05-05 22:49:11.574771 (MainThread): Parsing macros/materializations/view/create_or_replace_view.sql
2020-05-05 22:49:11.579251 (MainThread): Parsing macros/etc/get_custom_alias.sql
2020-05-05 22:49:11.580100 (MainThread): Parsing macros/etc/query.sql
2020-05-05 22:49:11.581074 (MainThread): Parsing macros/etc/is_incremental.sql
2020-05-05 22:49:11.582566 (MainThread): Parsing macros/etc/get_relation_comment.sql
2020-05-05 22:49:11.584420 (MainThread): Parsing macros/etc/datetime.sql
2020-05-05 22:49:11.592435 (MainThread): Parsing macros/etc/get_custom_schema.sql
2020-05-05 22:49:11.594204 (MainThread): Parsing macros/etc/get_custom_database.sql
2020-05-05 22:49:11.595153 (MainThread): Parsing macros/adapters/common.sql
2020-05-05 22:49:11.636189 (MainThread): Parsing macros/schema_tests/relationships.sql
2020-05-05 22:49:11.637285 (MainThread): Parsing macros/schema_tests/not_null.sql
2020-05-05 22:49:11.638123 (MainThread): Parsing macros/schema_tests/unique.sql
2020-05-05 22:49:11.639107 (MainThread): Parsing macros/schema_tests/accepted_values.sql
2020-05-05 22:49:11.641241 (MainThread): Parsing macros/etc.sql
2020-05-05 22:49:11.641835 (MainThread): Parsing macros/catalog.sql
2020-05-05 22:49:11.648127 (MainThread): Parsing macros/adapters.sql
2020-05-05 22:49:11.667045 (MainThread): Parsing macros/materializations/seed.sql
2020-05-05 22:49:11.668831 (MainThread): Parsing macros/materializations/view.sql
2020-05-05 22:49:11.670137 (MainThread): Parsing macros/materializations/table.sql
2020-05-05 22:49:11.679105 (MainThread): Parsing macros/materializations/incremental.sql
2020-05-05 22:49:11.690600 (MainThread): Parsing macros/materializations/snapshot.sql
2020-05-05 22:49:11.706375 (MainThread): Partial parsing not enabled
2020-05-05 22:49:11.733994 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-05-05 22:49:11.734079 (MainThread): Opening a new connection, currently in state init
2020-05-05 22:49:11.751000 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-05-05 22:49:11.751079 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:49:11.755606 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-05-05 22:49:11.755681 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:49:11.760213 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-05-05 22:49:11.760289 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:49:11.789104 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_account_account_id".
2020-05-05 22:49:11.789183 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:49:11.797126 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_account_account_id".
2020-05-05 22:49:11.797202 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:49:11.804667 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:49:11.804744 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:49:11.810580 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_opportunity_opportunity_id".
2020-05-05 22:49:11.810656 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:49:11.816523 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_user_id".
2020-05-05 22:49:11.816598 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:49:11.822519 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_user_id".
2020-05-05 22:49:11.822598 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:49:11.828776 (MainThread): Acquiring new bigquery connection "test.salesforce_source.not_null_stg_salesforce_user_role_user_role_id".
2020-05-05 22:49:11.828852 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:49:11.834710 (MainThread): Acquiring new bigquery connection "test.salesforce_source.unique_stg_salesforce_user_role_user_role_id".
2020-05-05 22:49:11.834788 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:49:12.121398 (MainThread): Found 4 models, 8 tests, 0 snapshots, 0 analyses, 141 macros, 0 operations, 0 seed files, 4 sources
2020-05-05 22:49:12.129409 (MainThread): 
2020-05-05 22:49:12.129784 (MainThread): Acquiring new bigquery connection "master".
2020-05-05 22:49:12.129968 (MainThread): Opening a new connection, currently in state closed
2020-05-05 22:49:12.663966 (ThreadPoolExecutor-1_0): Acquiring new bigquery connection "list_digital-arbor-400_dbt_kristin_test".
2020-05-05 22:49:12.664310 (ThreadPoolExecutor-1_0): Opening a new connection, currently in state init
2020-05-05 22:49:12.664899 (ThreadPoolExecutor-1_0): Client.dataset is deprecated and will be removed in a future version. Use a string like 'my_project.my_dataset' or a cloud.google.bigquery.DatasetReference object, instead.
2020-05-05 22:49:13.147804 (MainThread): 15:49:13 | Concurrency: 4 threads (target='dev')
2020-05-05 22:49:13.148015 (MainThread): 15:49:13 | 
2020-05-05 22:49:13.149888 (Thread-1): Began running node model.salesforce_source.stg_salesforce_account
2020-05-05 22:49:13.150239 (Thread-1): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-05-05 22:49:13.150367 (Thread-1): Re-using an available connection from the pool (formerly list_digital-arbor-400_dbt_kristin_test).
2020-05-05 22:49:13.150493 (Thread-1): Compiling model.salesforce_source.stg_salesforce_account
2020-05-05 22:49:13.155805 (Thread-2): Began running node model.salesforce_source.stg_salesforce_opportunity
2020-05-05 22:49:13.156146 (Thread-2): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-05-05 22:49:13.156273 (Thread-2): Opening a new connection, currently in state init
2020-05-05 22:49:13.156385 (Thread-2): Compiling model.salesforce_source.stg_salesforce_opportunity
2020-05-05 22:49:13.174088 (Thread-3): Began running node model.salesforce_source.stg_salesforce_user
2020-05-05 22:49:13.186919 (Thread-4): Began running node model.salesforce_source.stg_salesforce_user_role
2020-05-05 22:49:13.187250 (Thread-3): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-05-05 22:49:13.189568 (Thread-1): Writing injected SQL for node "model.salesforce_source.stg_salesforce_account"
2020-05-05 22:49:13.193362 (Thread-4): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-05-05 22:49:13.193645 (Thread-3): Opening a new connection, currently in state init
2020-05-05 22:49:13.193961 (Thread-2): finished collecting timing info
2020-05-05 22:49:13.194309 (Thread-4): Opening a new connection, currently in state init
2020-05-05 22:49:13.194519 (Thread-3): Compiling model.salesforce_source.stg_salesforce_user
2020-05-05 22:49:13.195010 (Thread-1): finished collecting timing info
2020-05-05 22:49:13.195394 (Thread-4): Compiling model.salesforce_source.stg_salesforce_user_role
2020-05-05 22:49:13.202788 (Thread-3): Writing injected SQL for node "model.salesforce_source.stg_salesforce_user"
2020-05-05 22:49:13.203065 (Thread-1): Finished running node model.salesforce_source.stg_salesforce_account
2020-05-05 22:49:13.209929 (Thread-4): Writing injected SQL for node "model.salesforce_source.stg_salesforce_user_role"
2020-05-05 22:49:13.210537 (Thread-3): finished collecting timing info
2020-05-05 22:49:13.211482 (Thread-3): Finished running node model.salesforce_source.stg_salesforce_user
2020-05-05 22:49:13.211748 (Thread-4): finished collecting timing info
2020-05-05 22:49:13.212089 (Thread-4): Finished running node model.salesforce_source.stg_salesforce_user_role
2020-05-05 22:49:13.194785 (Thread-2): Compilation Error in model stg_salesforce_opportunity (models/base/stg_salesforce_opportunity.sql)
  'dbt_utils' is undefined
Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 386, in catch_jinja
    yield
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 409, in render_template
    return template.render(ctx)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "<template>", line 22, in top-level template code
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/jinja2/sandbox.py", line 407, in getattr
    value = getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'dbt_utils' is undefined

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/node_runners.py", line 223, in safe_run
    result = self.compile_and_execute(manifest, ctx)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/node_runners.py", line 160, in compile_and_execute
    ctx.node = self.compile(manifest)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/node_runners.py", line 335, in compile
    return compile_node(self.adapter, self.config, self.node, manifest, {})
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/compilation.py", line 250, in compile_node
    node = compiler.compile_node(node, manifest, extra_context)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/compilation.py", line 157, in compile_node
    node)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 415, in get_rendered
    return render_template(template, ctx, node)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 409, in render_template
    return template.render(ctx)
  File "/usr/local/opt/python/Frameworks/Python.framework/Versions/3.7/lib/python3.7/contextlib.py", line 130, in __exit__
    self.gen.throw(type, value, traceback)
  File "/usr/local/Cellar/dbt/0.16.0_1/libexec/lib/python3.7/site-packages/dbt/clients/jinja.py", line 391, in catch_jinja
    raise CompilationException(str(e), node) from e
dbt.exceptions.CompilationException: Compilation Error in model stg_salesforce_opportunity (models/base/stg_salesforce_opportunity.sql)
  'dbt_utils' is undefined
2020-05-05 22:49:13.212625 (Thread-2): Finished running node model.salesforce_source.stg_salesforce_opportunity
2020-05-05 22:49:13.251843 (MainThread): 15:49:13 | 
2020-05-05 22:49:13.252129 (MainThread): 15:49:13 | Finished running  in 1.12s.
2020-05-05 22:49:13.252316 (MainThread): Connection 'master' was left open.
2020-05-05 22:49:13.252456 (MainThread): Connection 'model.salesforce_source.stg_salesforce_account' was left open.
2020-05-05 22:49:13.252587 (MainThread): Connection 'model.salesforce_source.stg_salesforce_opportunity' was properly closed.
2020-05-05 22:49:13.252714 (MainThread): Connection 'model.salesforce_source.stg_salesforce_user' was properly closed.
2020-05-05 22:49:13.252838 (MainThread): Connection 'model.salesforce_source.stg_salesforce_user_role' was properly closed.
2020-05-05 22:49:13.253645 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105909cd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1058fe390>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10592a1d0>]}
2020-05-05 22:49:13.253904 (MainThread): Flushing usage events
2020-10-23 18:10:11.961048 (MainThread): Running with dbt=0.18.1
2020-10-23 18:10:12.145945 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, defer=None, exclude=None, fail_fast=False, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='/Users/kristin/.dbt', project_dir=None, record_timing_info=None, rpc_method='run', selector_name=None, single_threaded=False, state=None, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, use_colors=None, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2020-10-23 18:10:12.147124 (MainThread): Tracking: tracking
2020-10-23 18:10:12.153457 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd9ab3f79a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd9ab424910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd9ab4248e0>]}
2020-10-23 18:10:12.175734 (MainThread): Partial parsing not enabled
2020-10-23 18:10:12.176695 (MainThread): Parsing macros/etc.sql
2020-10-23 18:10:12.178548 (MainThread): Parsing macros/catalog.sql
2020-10-23 18:10:12.185208 (MainThread): Parsing macros/adapters.sql
2020-10-23 18:10:12.205166 (MainThread): Parsing macros/materializations/seed.sql
2020-10-23 18:10:12.207719 (MainThread): Parsing macros/materializations/view.sql
2020-10-23 18:10:12.210194 (MainThread): Parsing macros/materializations/table.sql
2020-10-23 18:10:12.219339 (MainThread): Parsing macros/materializations/copy.sql
2020-10-23 18:10:12.223509 (MainThread): Parsing macros/materializations/incremental.sql
2020-10-23 18:10:12.235180 (MainThread): Parsing macros/materializations/snapshot.sql
2020-10-23 18:10:12.238167 (MainThread): Parsing macros/core.sql
2020-10-23 18:10:12.241457 (MainThread): Parsing macros/materializations/helpers.sql
2020-10-23 18:10:12.250040 (MainThread): Parsing macros/materializations/snapshot/snapshot_merge.sql
2020-10-23 18:10:12.251636 (MainThread): Parsing macros/materializations/snapshot/strategies.sql
2020-10-23 18:10:12.267252 (MainThread): Parsing macros/materializations/snapshot/snapshot.sql
2020-10-23 18:10:12.295232 (MainThread): Parsing macros/materializations/seed/seed.sql
2020-10-23 18:10:12.315039 (MainThread): Parsing macros/materializations/incremental/helpers.sql
2020-10-23 18:10:12.316957 (MainThread): Parsing macros/materializations/incremental/incremental.sql
2020-10-23 18:10:12.322589 (MainThread): Parsing macros/materializations/common/merge.sql
2020-10-23 18:10:12.335602 (MainThread): Parsing macros/materializations/table/table.sql
2020-10-23 18:10:12.342054 (MainThread): Parsing macros/materializations/view/view.sql
2020-10-23 18:10:12.347625 (MainThread): Parsing macros/materializations/view/create_or_replace_view.sql
2020-10-23 18:10:12.352207 (MainThread): Parsing macros/etc/get_custom_alias.sql
2020-10-23 18:10:12.353036 (MainThread): Parsing macros/etc/query.sql
2020-10-23 18:10:12.353961 (MainThread): Parsing macros/etc/is_incremental.sql
2020-10-23 18:10:12.355450 (MainThread): Parsing macros/etc/datetime.sql
2020-10-23 18:10:12.363426 (MainThread): Parsing macros/etc/get_custom_schema.sql
2020-10-23 18:10:12.365404 (MainThread): Parsing macros/etc/get_custom_database.sql
2020-10-23 18:10:12.366874 (MainThread): Parsing macros/adapters/common.sql
2020-10-23 18:10:12.405909 (MainThread): Parsing macros/schema_tests/relationships.sql
2020-10-23 18:10:12.407717 (MainThread): Parsing macros/schema_tests/not_null.sql
2020-10-23 18:10:12.409050 (MainThread): Parsing macros/schema_tests/unique.sql
2020-10-23 18:10:12.410588 (MainThread): Parsing macros/schema_tests/accepted_values.sql
2020-10-23 18:10:12.418015 (MainThread): Partial parsing not enabled
2020-10-23 18:10:12.482306 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-10-23 18:10:12.506041 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-10-23 18:10:12.515098 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-10-23 18:10:12.524501 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-10-23 18:10:12.940459 (MainThread): Found 4 models, 16 tests, 0 snapshots, 0 analyses, 155 macros, 0 operations, 0 seed files, 4 sources
2020-10-23 18:10:12.941449 (MainThread): 
2020-10-23 18:10:12.941846 (MainThread): Acquiring new bigquery connection "master".
2020-10-23 18:10:12.945971 (ThreadPoolExecutor-0_0): Acquiring new bigquery connection "list_digital-arbor-400".
2020-10-23 18:10:12.946112 (ThreadPoolExecutor-0_0): Opening a new connection, currently in state init
2020-10-23 18:10:12.947800 (ThreadPoolExecutor-0_0): Your application has authenticated using end user credentials from Google Cloud SDK. We recommend that most server applications use service accounts instead. If your application continues to use end user credentials from Cloud SDK, you might receive a "quota exceeded" or "API not enabled" error. For more information about service accounts, see https://cloud.google.com/docs/authentication/
2020-10-23 18:10:15.055795 (ThreadPoolExecutor-1_0): Acquiring new bigquery connection "list_digital-arbor-400_dbt_kristin_marketo".
2020-10-23 18:10:15.056090 (ThreadPoolExecutor-1_0): Opening a new connection, currently in state closed
2020-10-23 18:10:15.056463 (ThreadPoolExecutor-1_0): Your application has authenticated using end user credentials from Google Cloud SDK. We recommend that most server applications use service accounts instead. If your application continues to use end user credentials from Cloud SDK, you might receive a "quota exceeded" or "API not enabled" error. For more information about service accounts, see https://cloud.google.com/docs/authentication/
2020-10-23 18:10:16.563040 (ThreadPoolExecutor-1_0): Client.dataset is deprecated and will be removed in a future version. Use a string like 'my_project.my_dataset' or a cloud.google.bigquery.DatasetReference object, instead.
2020-10-23 18:10:16.965229 (MainThread): 11:10:16 | Concurrency: 4 threads (target='dev')
2020-10-23 18:10:16.965453 (MainThread): 11:10:16 | 
2020-10-23 18:10:16.969259 (Thread-1): Began running node model.salesforce_source.stg_salesforce_account
2020-10-23 18:10:16.970460 (Thread-1): 11:10:16 | 1 of 4 START table model dbt_kristin_marketo.stg_salesforce_account.. [RUN]
2020-10-23 18:10:16.970796 (Thread-1): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-10-23 18:10:16.970961 (Thread-1): Compiling model.salesforce_source.stg_salesforce_account
2020-10-23 18:10:16.976152 (Thread-2): Began running node model.salesforce_source.stg_salesforce_opportunity
2020-10-23 18:10:16.977302 (Thread-2): 11:10:16 | 2 of 4 START table model dbt_kristin_marketo.stg_salesforce_opportunity [RUN]
2020-10-23 18:10:16.983021 (Thread-2): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-10-23 18:10:16.983221 (Thread-3): Began running node model.salesforce_source.stg_salesforce_user
2020-10-23 18:10:16.983371 (Thread-2): Compiling model.salesforce_source.stg_salesforce_opportunity
2020-10-23 18:10:16.990224 (Thread-1): unclosed <socket.socket fd=19, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('192.168.4.24', 55896), raddr=('216.58.217.42', 443)>
2020-10-23 18:10:16.990407 (Thread-4): Began running node model.salesforce_source.stg_salesforce_user_role
2020-10-23 18:10:16.991633 (Thread-3): 11:10:16 | 3 of 4 START table model dbt_kristin_marketo.stg_salesforce_user..... [RUN]
2020-10-23 18:10:17.005974 (Thread-1): unclosed <socket.socket fd=20, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('192.168.4.24', 55897), raddr=('142.250.107.95', 443)>
2020-10-23 18:10:17.007238 (Thread-4): 11:10:17 | 4 of 4 START table model dbt_kristin_marketo.stg_salesforce_user_role [RUN]
2020-10-23 18:10:17.007817 (Thread-2): finished collecting timing info
2020-10-23 18:10:17.008279 (Thread-3): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-10-23 18:10:17.008487 (Thread-1): unclosed <socket.socket fd=21, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('192.168.4.24', 55902), raddr=('216.58.217.42', 443)>
2020-10-23 18:10:17.008816 (Thread-4): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-10-23 18:10:17.009400 (Thread-3): Compiling model.salesforce_source.stg_salesforce_user
2020-10-23 18:10:17.009838 (Thread-1): unclosed <socket.socket fd=22, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('192.168.4.24', 55903), raddr=('142.250.107.95', 443)>
2020-10-23 18:10:17.009968 (Thread-4): Compiling model.salesforce_source.stg_salesforce_user_role
2020-10-23 18:10:17.018931 (Thread-3): Writing injected SQL for node "model.salesforce_source.stg_salesforce_user"
2020-10-23 18:10:17.031284 (Thread-1): Writing injected SQL for node "model.salesforce_source.stg_salesforce_account"
2020-10-23 18:10:17.034919 (Thread-4): Writing injected SQL for node "model.salesforce_source.stg_salesforce_user_role"
2020-10-23 18:10:17.036542 (Thread-3): finished collecting timing info
2020-10-23 18:10:17.042860 (Thread-1): finished collecting timing info
2020-10-23 18:10:17.049010 (Thread-4): finished collecting timing info
2020-10-23 18:10:17.083674 (Thread-3): Opening a new connection, currently in state init
2020-10-23 18:10:17.084965 (Thread-1): Opening a new connection, currently in state closed
2020-10-23 18:10:17.085353 (Thread-4): Opening a new connection, currently in state init
2020-10-23 18:10:17.009100 (Thread-2): Compilation Error in model stg_salesforce_opportunity (models/base/stg_salesforce_opportunity.sql)
  'dbt_utils' is undefined
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/dbt/clients/jinja.py", line 496, in catch_jinja
    yield
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/dbt/clients/jinja.py", line 528, in render_template
    return template.render(ctx)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "<template>", line 22, in top-level template code
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/jinja2/sandbox.py", line 407, in getattr
    value = getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'dbt_utils' is undefined

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/dbt/task/base.py", line 333, in safe_run
    result = self.compile_and_execute(manifest, ctx)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/dbt/task/base.py", line 270, in compile_and_execute
    ctx.node = self.compile(manifest)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/dbt/task/compile.py", line 23, in compile
    return compiler.compile_node(self.node, manifest, {})
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/dbt/compilation.py", line 479, in compile_node
    node = self._compile_node(node, manifest, extra_context)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/dbt/compilation.py", line 385, in _compile_node
    compiled_node.compiled_sql = jinja.get_rendered(
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/dbt/clients/jinja.py", line 577, in get_rendered
    return render_template(template, ctx, node)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/dbt/clients/jinja.py", line 528, in render_template
    return template.render(ctx)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/contextlib.py", line 131, in __exit__
    self.gen.throw(type, value, traceback)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/dbt/clients/jinja.py", line 501, in catch_jinja
    raise CompilationException(str(e), node) from e
dbt.exceptions.CompilationException: Compilation Error in model stg_salesforce_opportunity (models/base/stg_salesforce_opportunity.sql)
  'dbt_utils' is undefined
2020-10-23 18:10:17.086136 (Thread-2): Sending event: {'category': 'dbt', 'action': 'run_model', 'label': '8f5709ac-ab34-4be2-bd0a-fb7f698f0d26', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd9ab7055b0>]}
2020-10-23 18:10:17.087011 (Thread-2): 11:10:17 | 2 of 4 ERROR creating table model dbt_kristin_marketo.stg_salesforce_opportunity [ERROR in 0.10s]
2020-10-23 18:10:17.087151 (Thread-4): Your application has authenticated using end user credentials from Google Cloud SDK. We recommend that most server applications use service accounts instead. If your application continues to use end user credentials from Cloud SDK, you might receive a "quota exceeded" or "API not enabled" error. For more information about service accounts, see https://cloud.google.com/docs/authentication/
2020-10-23 18:10:17.087418 (Thread-2): Finished running node model.salesforce_source.stg_salesforce_opportunity
2020-10-23 18:10:17.087769 (Thread-3): Your application has authenticated using end user credentials from Google Cloud SDK. We recommend that most server applications use service accounts instead. If your application continues to use end user credentials from Cloud SDK, you might receive a "quota exceeded" or "API not enabled" error. For more information about service accounts, see https://cloud.google.com/docs/authentication/
2020-10-23 18:10:17.093162 (Thread-1): Your application has authenticated using end user credentials from Google Cloud SDK. We recommend that most server applications use service accounts instead. If your application continues to use end user credentials from Cloud SDK, you might receive a "quota exceeded" or "API not enabled" error. For more information about service accounts, see https://cloud.google.com/docs/authentication/
2020-10-23 18:10:18.794249 (Thread-1): Client.dataset is deprecated and will be removed in a future version. Use a string like 'my_project.my_dataset' or a cloud.google.bigquery.DatasetReference object, instead.
2020-10-23 18:10:18.796018 (Thread-4): Client.dataset is deprecated and will be removed in a future version. Use a string like 'my_project.my_dataset' or a cloud.google.bigquery.DatasetReference object, instead.
2020-10-23 18:10:18.797987 (Thread-3): Client.dataset is deprecated and will be removed in a future version. Use a string like 'my_project.my_dataset' or a cloud.google.bigquery.DatasetReference object, instead.
2020-10-23 18:10:19.206025 (Thread-4): Writing runtime SQL for node "model.salesforce_source.stg_salesforce_user_role"
2020-10-23 18:10:19.207208 (Thread-3): Writing runtime SQL for node "model.salesforce_source.stg_salesforce_user"
2020-10-23 18:10:19.208135 (Thread-1): Writing runtime SQL for node "model.salesforce_source.stg_salesforce_account"
2020-10-23 18:10:19.208964 (Thread-4): On model.salesforce_source.stg_salesforce_user_role: /* {"app": "dbt", "dbt_version": "0.18.1", "profile_name": "fivetran", "target_name": "dev", "node_id": "model.salesforce_source.stg_salesforce_user_role"} */


  create or replace table `digital-arbor-400`.`dbt_kristin_marketo`.`stg_salesforce_user_role`
  
  
  OPTIONS()
  as (
    with base as (

    select *
    from `digital-arbor-400`.`salesforce`.`user_role`
    where not _fivetran_deleted

), fields as (

    select 

      id as user_role_id,
      name as role_name

    from base

)

select *
from fields
  );
    
2020-10-23 18:10:19.209652 (Thread-1): On model.salesforce_source.stg_salesforce_account: /* {"app": "dbt", "dbt_version": "0.18.1", "profile_name": "fivetran", "target_name": "dev", "node_id": "model.salesforce_source.stg_salesforce_account"} */


  create or replace table `digital-arbor-400`.`dbt_kristin_marketo`.`stg_salesforce_account`
  
  
  OPTIONS()
  as (
    with base as (

    select *
    from `digital-arbor-400`.`salesforce`.`account`
    where not is_deleted

), fields as (

    select 

      id as account_id,
      name as account_name,
      industry,
      number_of_employees,
      account_source

    from base

)

select *
from fields
  );
    
2020-10-23 18:10:19.210556 (Thread-3): On model.salesforce_source.stg_salesforce_user: /* {"app": "dbt", "dbt_version": "0.18.1", "profile_name": "fivetran", "target_name": "dev", "node_id": "model.salesforce_source.stg_salesforce_user"} */


  create or replace table `digital-arbor-400`.`dbt_kristin_marketo`.`stg_salesforce_user`
  
  
  OPTIONS()
  as (
    with base as (

    select *
    from `digital-arbor-400`.`salesforce`.`user`

), fields as (

    select 
      id as user_id,
      name,
      city,
      state,
      manager_id,
      user_role_id
    from base

)

select *
from fields
  );
    
2020-10-23 18:10:21.121745 (Thread-4): finished collecting timing info
2020-10-23 18:10:21.123148 (Thread-4): Sending event: {'category': 'dbt', 'action': 'run_model', 'label': '8f5709ac-ab34-4be2-bd0a-fb7f698f0d26', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd9ab6b2970>]}
2020-10-23 18:10:21.124788 (Thread-4): 11:10:21 | 4 of 4 OK created table model dbt_kristin_marketo.stg_salesforce_user_role [CREATE TABLE (96.0 rows, 4.5 KB processed) in 4.11s]
2020-10-23 18:10:21.124979 (Thread-4): Finished running node model.salesforce_source.stg_salesforce_user_role
2020-10-23 18:10:21.141063 (Thread-3): finished collecting timing info
2020-10-23 18:10:21.142080 (Thread-3): Sending event: {'category': 'dbt', 'action': 'run_model', 'label': '8f5709ac-ab34-4be2-bd0a-fb7f698f0d26', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd9ab6d3df0>]}
2020-10-23 18:10:21.143248 (Thread-3): 11:10:21 | 3 of 4 OK created table model dbt_kristin_marketo.stg_salesforce_user [CREATE TABLE (376.0 rows, 26.0 KB processed) in 4.13s]
2020-10-23 18:10:21.143402 (Thread-3): Finished running node model.salesforce_source.stg_salesforce_user
2020-10-23 18:10:23.589187 (Thread-1): finished collecting timing info
2020-10-23 18:10:23.590031 (Thread-1): Sending event: {'category': 'dbt', 'action': 'run_model', 'label': '8f5709ac-ab34-4be2-bd0a-fb7f698f0d26', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd9ab9e2820>]}
2020-10-23 18:10:23.591336 (Thread-1): 11:10:23 | 1 of 4 OK created table model dbt_kristin_marketo.stg_salesforce_account [CREATE TABLE (265.4k rows, 27.5 MB processed) in 6.62s]
2020-10-23 18:10:23.591509 (Thread-1): Finished running node model.salesforce_source.stg_salesforce_account
2020-10-23 18:10:23.593078 (MainThread): Acquiring new bigquery connection "master".
2020-10-23 18:10:23.593445 (MainThread): 11:10:23 | 
2020-10-23 18:10:23.593589 (MainThread): 11:10:23 | Finished running 4 table models in 10.65s.
2020-10-23 18:10:23.593704 (MainThread): Connection 'master' was properly closed.
2020-10-23 18:10:23.593789 (MainThread): Connection 'model.salesforce_source.stg_salesforce_account' was properly closed.
2020-10-23 18:10:23.593872 (MainThread): Connection 'model.salesforce_source.stg_salesforce_opportunity' was properly closed.
2020-10-23 18:10:23.593950 (MainThread): Connection 'model.salesforce_source.stg_salesforce_user' was properly closed.
2020-10-23 18:10:23.594026 (MainThread): Connection 'model.salesforce_source.stg_salesforce_user_role' was properly closed.
2020-10-23 18:10:23.607049 (MainThread): 
2020-10-23 18:10:23.607234 (MainThread): Completed with 1 error and 0 warnings:
2020-10-23 18:10:23.607354 (MainThread): 
2020-10-23 18:10:23.607470 (MainThread): Compilation Error in model stg_salesforce_opportunity (models/base/stg_salesforce_opportunity.sql)
2020-10-23 18:10:23.607573 (MainThread):   'dbt_utils' is undefined
2020-10-23 18:10:23.607676 (MainThread): 
Done. PASS=3 WARN=0 ERROR=1 SKIP=0 TOTAL=4
2020-10-23 18:10:23.607888 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd9ab5c6160>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd9ab59d8e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd9ab487ca0>]}
2020-10-23 18:10:23.608099 (MainThread): Flushing usage events
2020-10-23 18:10:28.292213 (MainThread): Running with dbt=0.18.1
2020-10-23 18:10:28.491128 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.deps.DepsTask'>, debug=False, defer=None, log_cache_events=False, log_format='default', partial_parse=None, profile=None, profiles_dir='/Users/kristin/.dbt', project_dir=None, record_timing_info=None, rpc_method='deps', single_threaded=False, state=None, strict=False, target=None, test_new_parser=False, use_cache=True, use_colors=None, vars='{}', warn_error=False, which='deps', write_json=True)
2020-10-23 18:10:28.491438 (MainThread): Tracking: tracking
2020-10-23 18:10:28.496882 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fb80f6d5ee0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fb80fd2dee0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fb80fd2dfa0>]}
2020-10-23 18:10:28.498469 (MainThread): Set downloads directory='/var/folders/sk/bwywl8692vlbvlbk71j59w780000gp/T/dbt-downloads-iyv0twf8'
2020-10-23 18:10:28.500027 (MainThread): Making package registry request: GET https://hub.getdbt.com/api/v1/index.json
2020-10-23 18:10:28.823390 (MainThread): Response from registry: GET https://hub.getdbt.com/api/v1/index.json 200
2020-10-23 18:10:28.823711 (MainThread): Making package registry request: GET https://hub.getdbt.com/api/v1/fishtown-analytics/dbt_utils.json
2020-10-23 18:10:29.104536 (MainThread): Response from registry: GET https://hub.getdbt.com/api/v1/fishtown-analytics/dbt_utils.json 200
2020-10-23 18:10:29.119967 (MainThread): Making package registry request: GET https://hub.getdbt.com/api/v1/fishtown-analytics/dbt_utils/0.6.2.json
2020-10-23 18:10:29.464118 (MainThread): Response from registry: GET https://hub.getdbt.com/api/v1/fishtown-analytics/dbt_utils/0.6.2.json 200
2020-10-23 18:10:29.469202 (MainThread): Making package registry request: GET https://hub.getdbt.com/api/v1/fishtown-analytics/dbt_utils.json
2020-10-23 18:10:29.760056 (MainThread): Response from registry: GET https://hub.getdbt.com/api/v1/fishtown-analytics/dbt_utils.json 200
2020-10-23 18:10:29.775748 (MainThread): Making package registry request: GET https://hub.getdbt.com/api/v1/fishtown-analytics/dbt_utils/0.6.2.json
2020-10-23 18:10:30.109273 (MainThread): Response from registry: GET https://hub.getdbt.com/api/v1/fishtown-analytics/dbt_utils/0.6.2.json 200
2020-10-23 18:10:30.110383 (MainThread): Installing fishtown-analytics/dbt_utils@0.6.2
2020-10-23 18:10:30.670832 (MainThread):   Installed from version 0.6.2

2020-10-23 18:10:30.671150 (MainThread): Sending event: {'category': 'dbt', 'action': 'package', 'label': 'cb63eef5-b337-4ce1-896a-146ab5dc741d', 'property_': 'install', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fb80fd57520>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fb80fd575e0>]}
2020-10-23 18:10:30.672086 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fb80f6d5ee0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fb80fd57430>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fb80fd577f0>]}
2020-10-23 18:10:30.672435 (MainThread): Flushing usage events
2020-10-23 18:10:37.026697 (MainThread): Running with dbt=0.18.1
2020-10-23 18:10:37.210601 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, defer=None, exclude=None, fail_fast=False, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='/Users/kristin/.dbt', project_dir=None, record_timing_info=None, rpc_method='run', selector_name=None, single_threaded=False, state=None, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, use_colors=None, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2020-10-23 18:10:37.211762 (MainThread): Tracking: tracking
2020-10-23 18:10:37.217825 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7feef162f160>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7feef1636880>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7feef1636850>]}
2020-10-23 18:10:37.248882 (MainThread): Partial parsing not enabled
2020-10-23 18:10:37.249818 (MainThread): Parsing macros/etc.sql
2020-10-23 18:10:37.251438 (MainThread): Parsing macros/catalog.sql
2020-10-23 18:10:37.256883 (MainThread): Parsing macros/adapters.sql
2020-10-23 18:10:37.274272 (MainThread): Parsing macros/materializations/seed.sql
2020-10-23 18:10:37.276822 (MainThread): Parsing macros/materializations/view.sql
2020-10-23 18:10:37.279459 (MainThread): Parsing macros/materializations/table.sql
2020-10-23 18:10:37.289074 (MainThread): Parsing macros/materializations/copy.sql
2020-10-23 18:10:37.292967 (MainThread): Parsing macros/materializations/incremental.sql
2020-10-23 18:10:37.303817 (MainThread): Parsing macros/materializations/snapshot.sql
2020-10-23 18:10:37.306300 (MainThread): Parsing macros/core.sql
2020-10-23 18:10:37.309590 (MainThread): Parsing macros/materializations/helpers.sql
2020-10-23 18:10:37.317550 (MainThread): Parsing macros/materializations/snapshot/snapshot_merge.sql
2020-10-23 18:10:37.319096 (MainThread): Parsing macros/materializations/snapshot/strategies.sql
2020-10-23 18:10:37.334310 (MainThread): Parsing macros/materializations/snapshot/snapshot.sql
2020-10-23 18:10:37.362611 (MainThread): Parsing macros/materializations/seed/seed.sql
2020-10-23 18:10:37.382490 (MainThread): Parsing macros/materializations/incremental/helpers.sql
2020-10-23 18:10:37.384215 (MainThread): Parsing macros/materializations/incremental/incremental.sql
2020-10-23 18:10:37.389981 (MainThread): Parsing macros/materializations/common/merge.sql
2020-10-23 18:10:37.403181 (MainThread): Parsing macros/materializations/table/table.sql
2020-10-23 18:10:37.409328 (MainThread): Parsing macros/materializations/view/view.sql
2020-10-23 18:10:37.414751 (MainThread): Parsing macros/materializations/view/create_or_replace_view.sql
2020-10-23 18:10:37.419100 (MainThread): Parsing macros/etc/get_custom_alias.sql
2020-10-23 18:10:37.419906 (MainThread): Parsing macros/etc/query.sql
2020-10-23 18:10:37.420808 (MainThread): Parsing macros/etc/is_incremental.sql
2020-10-23 18:10:37.422196 (MainThread): Parsing macros/etc/datetime.sql
2020-10-23 18:10:37.429897 (MainThread): Parsing macros/etc/get_custom_schema.sql
2020-10-23 18:10:37.431579 (MainThread): Parsing macros/etc/get_custom_database.sql
2020-10-23 18:10:37.433019 (MainThread): Parsing macros/adapters/common.sql
2020-10-23 18:10:37.471927 (MainThread): Parsing macros/schema_tests/relationships.sql
2020-10-23 18:10:37.473573 (MainThread): Parsing macros/schema_tests/not_null.sql
2020-10-23 18:10:37.474869 (MainThread): Parsing macros/schema_tests/unique.sql
2020-10-23 18:10:37.476430 (MainThread): Parsing macros/schema_tests/accepted_values.sql
2020-10-23 18:10:37.480034 (MainThread): Parsing macros/cross_db_utils/except.sql
2020-10-23 18:10:37.481068 (MainThread): Parsing macros/cross_db_utils/replace.sql
2020-10-23 18:10:37.482320 (MainThread): Parsing macros/cross_db_utils/concat.sql
2020-10-23 18:10:37.484540 (MainThread): Parsing macros/cross_db_utils/datatypes.sql
2020-10-23 18:10:37.490387 (MainThread): Parsing macros/cross_db_utils/_is_relation.sql
2020-10-23 18:10:37.491566 (MainThread): Parsing macros/cross_db_utils/length.sql
2020-10-23 18:10:37.493363 (MainThread): Parsing macros/cross_db_utils/dateadd.sql
2020-10-23 18:10:37.495708 (MainThread): Parsing macros/cross_db_utils/intersect.sql
2020-10-23 18:10:37.496815 (MainThread): Parsing macros/cross_db_utils/right.sql
2020-10-23 18:10:37.499157 (MainThread): Parsing macros/cross_db_utils/datediff.sql
2020-10-23 18:10:37.532969 (MainThread): Parsing macros/cross_db_utils/safe_cast.sql
2020-10-23 18:10:37.535786 (MainThread): Parsing macros/cross_db_utils/hash.sql
2020-10-23 18:10:37.537800 (MainThread): Parsing macros/cross_db_utils/identifier.sql
2020-10-23 18:10:37.540160 (MainThread): Parsing macros/cross_db_utils/position.sql
2020-10-23 18:10:37.542350 (MainThread): Parsing macros/cross_db_utils/literal.sql
2020-10-23 18:10:37.543781 (MainThread): Parsing macros/cross_db_utils/current_timestamp.sql
2020-10-23 18:10:37.548398 (MainThread): Parsing macros/cross_db_utils/width_bucket.sql
2020-10-23 18:10:37.555323 (MainThread): Parsing macros/cross_db_utils/_get_utils_namespaces.sql
2020-10-23 18:10:37.556368 (MainThread): Parsing macros/cross_db_utils/last_day.sql
2020-10-23 18:10:37.559666 (MainThread): Parsing macros/cross_db_utils/split_part.sql
2020-10-23 18:10:37.561877 (MainThread): Parsing macros/cross_db_utils/date_trunc.sql
2020-10-23 18:10:37.563716 (MainThread): Parsing macros/cross_db_utils/_is_ephemeral.sql
2020-10-23 18:10:37.566157 (MainThread): Parsing macros/materializations/insert_by_period_materialization.sql
2020-10-23 18:10:37.590651 (MainThread): Parsing macros/logger/pretty_log_format.sql
2020-10-23 18:10:37.591563 (MainThread): Parsing macros/logger/pretty_time.sql
2020-10-23 18:10:37.592263 (MainThread): Parsing macros/logger/log_info.sql
2020-10-23 18:10:37.592908 (MainThread): Parsing macros/datetime/date_spine.sql
2020-10-23 18:10:37.596582 (MainThread): Parsing macros/web/get_url_host.sql
2020-10-23 18:10:37.597869 (MainThread): Parsing macros/web/get_url_path.sql
2020-10-23 18:10:37.599734 (MainThread): Parsing macros/web/get_url_parameter.sql
2020-10-23 18:10:37.600682 (MainThread): Parsing macros/geo/haversine_distance.sql
2020-10-23 18:10:37.601479 (MainThread): Parsing macros/schema_tests/equal_rowcount.sql
2020-10-23 18:10:37.602593 (MainThread): Parsing macros/schema_tests/relationships_where.sql
2020-10-23 18:10:37.604456 (MainThread): Parsing macros/schema_tests/recency.sql
2020-10-23 18:10:37.605504 (MainThread): Parsing macros/schema_tests/not_constant.sql
2020-10-23 18:10:37.606349 (MainThread): Parsing macros/schema_tests/test_unique_where.sql
2020-10-23 18:10:37.607696 (MainThread): Parsing macros/schema_tests/at_least_one.sql
2020-10-23 18:10:37.608789 (MainThread): Parsing macros/schema_tests/unique_combination_of_columns.sql
2020-10-23 18:10:37.611099 (MainThread): Parsing macros/schema_tests/cardinality_equality.sql
2020-10-23 18:10:37.612393 (MainThread): Parsing macros/schema_tests/expression_is_true.sql
2020-10-23 18:10:37.613321 (MainThread): Parsing macros/schema_tests/test_not_null_where.sql
2020-10-23 18:10:37.614606 (MainThread): Parsing macros/schema_tests/equality.sql
2020-10-23 18:10:37.617453 (MainThread): Parsing macros/schema_tests/mutually_exclusive_ranges.sql
2020-10-23 18:10:37.623002 (MainThread): Parsing macros/sql/nullcheck_table.sql
2020-10-23 18:10:37.624876 (MainThread): Parsing macros/sql/get_relations_by_pattern.sql
2020-10-23 18:10:37.627868 (MainThread): Parsing macros/sql/generate_series.sql
2020-10-23 18:10:37.632465 (MainThread): Parsing macros/sql/get_relations_by_prefix.sql
2020-10-23 18:10:37.635327 (MainThread): Parsing macros/sql/get_tables_by_prefix_sql.sql
2020-10-23 18:10:37.636250 (MainThread): Parsing macros/sql/star.sql
2020-10-23 18:10:37.638893 (MainThread): Parsing macros/sql/unpivot.sql
2020-10-23 18:10:37.644622 (MainThread): Parsing macros/sql/union.sql
2020-10-23 18:10:37.651637 (MainThread): Parsing macros/sql/groupby.sql
2020-10-23 18:10:37.652497 (MainThread): Parsing macros/sql/surrogate_key.sql
2020-10-23 18:10:37.655078 (MainThread): Parsing macros/sql/safe_add.sql
2020-10-23 18:10:37.655986 (MainThread): Parsing macros/sql/nullcheck.sql
2020-10-23 18:10:37.657058 (MainThread): Parsing macros/sql/get_tables_by_pattern_sql.sql
2020-10-23 18:10:37.662539 (MainThread): Parsing macros/sql/get_column_values.sql
2020-10-23 18:10:37.666063 (MainThread): Parsing macros/sql/pivot.sql
2020-10-23 18:10:37.668516 (MainThread): Parsing macros/sql/get_query_results_as_dict.sql
2020-10-23 18:10:37.674624 (MainThread): Partial parsing not enabled
2020-10-23 18:10:37.710527 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-10-23 18:10:37.740266 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-10-23 18:10:37.749843 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-10-23 18:10:37.759600 (MainThread): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-10-23 18:10:38.292744 (MainThread): Found 4 models, 16 tests, 0 snapshots, 0 analyses, 287 macros, 0 operations, 0 seed files, 4 sources
2020-10-23 18:10:38.293969 (MainThread): 
2020-10-23 18:10:38.294311 (MainThread): Acquiring new bigquery connection "master".
2020-10-23 18:10:38.298254 (ThreadPoolExecutor-0_0): Acquiring new bigquery connection "list_digital-arbor-400".
2020-10-23 18:10:38.298377 (ThreadPoolExecutor-0_0): Opening a new connection, currently in state init
2020-10-23 18:10:38.300195 (ThreadPoolExecutor-0_0): Your application has authenticated using end user credentials from Google Cloud SDK. We recommend that most server applications use service accounts instead. If your application continues to use end user credentials from Cloud SDK, you might receive a "quota exceeded" or "API not enabled" error. For more information about service accounts, see https://cloud.google.com/docs/authentication/
2020-10-23 18:10:40.480458 (ThreadPoolExecutor-1_0): Acquiring new bigquery connection "list_digital-arbor-400_dbt_kristin_marketo".
2020-10-23 18:10:40.480733 (ThreadPoolExecutor-1_0): Opening a new connection, currently in state closed
2020-10-23 18:10:40.481086 (ThreadPoolExecutor-1_0): Your application has authenticated using end user credentials from Google Cloud SDK. We recommend that most server applications use service accounts instead. If your application continues to use end user credentials from Cloud SDK, you might receive a "quota exceeded" or "API not enabled" error. For more information about service accounts, see https://cloud.google.com/docs/authentication/
2020-10-23 18:10:41.705985 (ThreadPoolExecutor-1_0): Client.dataset is deprecated and will be removed in a future version. Use a string like 'my_project.my_dataset' or a cloud.google.bigquery.DatasetReference object, instead.
2020-10-23 18:10:42.105320 (MainThread): 11:10:42 | Concurrency: 4 threads (target='dev')
2020-10-23 18:10:42.105528 (MainThread): 11:10:42 | 
2020-10-23 18:10:42.107783 (MainThread): unclosed <socket.socket fd=19, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('192.168.4.24', 55933), raddr=('172.217.3.170', 443)>
2020-10-23 18:10:42.107964 (MainThread): unclosed <socket.socket fd=20, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('192.168.4.24', 55934), raddr=('142.250.107.95', 443)>
2020-10-23 18:10:42.109950 (Thread-1): Began running node model.salesforce_source.stg_salesforce_account
2020-10-23 18:10:42.111124 (Thread-1): 11:10:42 | 1 of 4 START table model dbt_kristin_marketo.stg_salesforce_account.. [RUN]
2020-10-23 18:10:42.111469 (Thread-1): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_account".
2020-10-23 18:10:42.111624 (Thread-1): Compiling model.salesforce_source.stg_salesforce_account
2020-10-23 18:10:42.117019 (Thread-2): Began running node model.salesforce_source.stg_salesforce_opportunity
2020-10-23 18:10:42.123213 (Thread-2): 11:10:42 | 2 of 4 START table model dbt_kristin_marketo.stg_salesforce_opportunity [RUN]
2020-10-23 18:10:42.123396 (Thread-3): Began running node model.salesforce_source.stg_salesforce_user
2020-10-23 18:10:42.124474 (Thread-3): 11:10:42 | 3 of 4 START table model dbt_kristin_marketo.stg_salesforce_user..... [RUN]
2020-10-23 18:10:42.124635 (Thread-4): Began running node model.salesforce_source.stg_salesforce_user_role
2020-10-23 18:10:42.125743 (Thread-4): 11:10:42 | 4 of 4 START table model dbt_kristin_marketo.stg_salesforce_user_role [RUN]
2020-10-23 18:10:42.131413 (Thread-2): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_opportunity".
2020-10-23 18:10:42.139848 (Thread-1): Writing injected SQL for node "model.salesforce_source.stg_salesforce_account"
2020-10-23 18:10:42.140137 (Thread-3): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user".
2020-10-23 18:10:42.140233 (Thread-2): Compiling model.salesforce_source.stg_salesforce_opportunity
2020-10-23 18:10:42.140450 (Thread-4): Acquiring new bigquery connection "model.salesforce_source.stg_salesforce_user_role".
2020-10-23 18:10:42.140771 (Thread-3): Compiling model.salesforce_source.stg_salesforce_user
2020-10-23 18:10:42.146073 (Thread-4): Compiling model.salesforce_source.stg_salesforce_user_role
2020-10-23 18:10:42.157567 (Thread-2): Writing injected SQL for node "model.salesforce_source.stg_salesforce_opportunity"
2020-10-23 18:10:42.157735 (Thread-1): finished collecting timing info
2020-10-23 18:10:42.167345 (Thread-3): Writing injected SQL for node "model.salesforce_source.stg_salesforce_user"
2020-10-23 18:10:42.176148 (Thread-4): Writing injected SQL for node "model.salesforce_source.stg_salesforce_user_role"
2020-10-23 18:10:42.198309 (Thread-1): Opening a new connection, currently in state closed
2020-10-23 18:10:42.198559 (Thread-2): finished collecting timing info
2020-10-23 18:10:42.202387 (Thread-2): Opening a new connection, currently in state init
2020-10-23 18:10:42.202621 (Thread-3): finished collecting timing info
2020-10-23 18:10:42.207163 (Thread-3): Opening a new connection, currently in state init
2020-10-23 18:10:42.207566 (Thread-1): Your application has authenticated using end user credentials from Google Cloud SDK. We recommend that most server applications use service accounts instead. If your application continues to use end user credentials from Cloud SDK, you might receive a "quota exceeded" or "API not enabled" error. For more information about service accounts, see https://cloud.google.com/docs/authentication/
2020-10-23 18:10:42.210116 (Thread-2): Your application has authenticated using end user credentials from Google Cloud SDK. We recommend that most server applications use service accounts instead. If your application continues to use end user credentials from Cloud SDK, you might receive a "quota exceeded" or "API not enabled" error. For more information about service accounts, see https://cloud.google.com/docs/authentication/
2020-10-23 18:10:42.211608 (Thread-3): Your application has authenticated using end user credentials from Google Cloud SDK. We recommend that most server applications use service accounts instead. If your application continues to use end user credentials from Cloud SDK, you might receive a "quota exceeded" or "API not enabled" error. For more information about service accounts, see https://cloud.google.com/docs/authentication/
2020-10-23 18:10:42.217526 (Thread-4): finished collecting timing info
2020-10-23 18:10:42.225936 (Thread-4): Opening a new connection, currently in state init
2020-10-23 18:10:42.226526 (Thread-4): Your application has authenticated using end user credentials from Google Cloud SDK. We recommend that most server applications use service accounts instead. If your application continues to use end user credentials from Cloud SDK, you might receive a "quota exceeded" or "API not enabled" error. For more information about service accounts, see https://cloud.google.com/docs/authentication/
2020-10-23 18:10:43.648426 (Thread-1): Client.dataset is deprecated and will be removed in a future version. Use a string like 'my_project.my_dataset' or a cloud.google.bigquery.DatasetReference object, instead.
2020-10-23 18:10:43.818687 (Thread-3): Client.dataset is deprecated and will be removed in a future version. Use a string like 'my_project.my_dataset' or a cloud.google.bigquery.DatasetReference object, instead.
2020-10-23 18:10:43.847761 (Thread-2): Client.dataset is deprecated and will be removed in a future version. Use a string like 'my_project.my_dataset' or a cloud.google.bigquery.DatasetReference object, instead.
2020-10-23 18:10:43.895572 (Thread-4): Client.dataset is deprecated and will be removed in a future version. Use a string like 'my_project.my_dataset' or a cloud.google.bigquery.DatasetReference object, instead.
2020-10-23 18:10:44.037244 (Thread-1): Writing runtime SQL for node "model.salesforce_source.stg_salesforce_account"
2020-10-23 18:10:44.039204 (Thread-1): On model.salesforce_source.stg_salesforce_account: /* {"app": "dbt", "dbt_version": "0.18.1", "profile_name": "fivetran", "target_name": "dev", "node_id": "model.salesforce_source.stg_salesforce_account"} */


  create or replace table `digital-arbor-400`.`dbt_kristin_marketo`.`stg_salesforce_account`
  
  
  OPTIONS()
  as (
    with base as (

    select *
    from `digital-arbor-400`.`salesforce`.`account`
    where not is_deleted

), fields as (

    select 

      id as account_id,
      name as account_name,
      industry,
      number_of_employees,
      account_source

    from base

)

select *
from fields
  );
    
2020-10-23 18:10:44.209717 (Thread-3): Writing runtime SQL for node "model.salesforce_source.stg_salesforce_user"
2020-10-23 18:10:44.210269 (Thread-3): On model.salesforce_source.stg_salesforce_user: /* {"app": "dbt", "dbt_version": "0.18.1", "profile_name": "fivetran", "target_name": "dev", "node_id": "model.salesforce_source.stg_salesforce_user"} */


  create or replace table `digital-arbor-400`.`dbt_kristin_marketo`.`stg_salesforce_user`
  
  
  OPTIONS()
  as (
    with base as (

    select *
    from `digital-arbor-400`.`salesforce`.`user`

), fields as (

    select 
      id as user_id,
      name,
      city,
      state,
      manager_id,
      user_role_id
    from base

)

select *
from fields
  );
    
2020-10-23 18:10:44.258689 (Thread-2): Writing runtime SQL for node "model.salesforce_source.stg_salesforce_opportunity"
2020-10-23 18:10:44.259384 (Thread-2): On model.salesforce_source.stg_salesforce_opportunity: /* {"app": "dbt", "dbt_version": "0.18.1", "profile_name": "fivetran", "target_name": "dev", "node_id": "model.salesforce_source.stg_salesforce_opportunity"} */


  create or replace table `digital-arbor-400`.`dbt_kristin_marketo`.`stg_salesforce_opportunity`
  
  
  OPTIONS()
  as (
    with base as (

    select *
    from `digital-arbor-400`.`salesforce`.`opportunity`
    where not is_deleted

), fields as (

    select 

        id as opportunity_id,
        account_id as opportunity_account_id,
        amount,
        probability,
        created_date, 
        is_won,
        is_closed,
        forecast_category,
        stage_name,
        owner_id,
        close_date,
        created_date >= 
    timestamp_trunc(
        cast(
    current_timestamp
 as timestamp),
        month
    )

 as is_created_this_month,
        created_date >= 
    timestamp_trunc(
        cast(
    current_timestamp
 as timestamp),
        quarter
    )

 as is_created_this_quarter,
        
  

    datetime_diff(
        cast(created_date as datetime),
        cast(
    current_timestamp
 as datetime),
        day
    )


 as days_since_created,
        
  

    datetime_diff(
        cast(created_date as datetime),
        cast(close_date as datetime),
        day
    )


 as days_to_close,
        
    timestamp_trunc(
        cast(close_date as timestamp),
        month
    )

 = 
    timestamp_trunc(
        cast(
    current_timestamp
 as timestamp),
        month
    )

 as is_closed_this_month,
        
    timestamp_trunc(
        cast(close_date as timestamp),
        quarter
    )

 = 
    timestamp_trunc(
        cast(
    current_timestamp
 as timestamp),
        quarter
    )

 as is_closed_this_quarter

    from base

)

select *
from fields
  );
    
2020-10-23 18:10:44.263794 (Thread-4): Writing runtime SQL for node "model.salesforce_source.stg_salesforce_user_role"
2020-10-23 18:10:44.264231 (Thread-4): On model.salesforce_source.stg_salesforce_user_role: /* {"app": "dbt", "dbt_version": "0.18.1", "profile_name": "fivetran", "target_name": "dev", "node_id": "model.salesforce_source.stg_salesforce_user_role"} */


  create or replace table `digital-arbor-400`.`dbt_kristin_marketo`.`stg_salesforce_user_role`
  
  
  OPTIONS()
  as (
    with base as (

    select *
    from `digital-arbor-400`.`salesforce`.`user_role`
    where not _fivetran_deleted

), fields as (

    select 

      id as user_role_id,
      name as role_name

    from base

)

select *
from fields
  );
    
2020-10-23 18:10:46.083357 (Thread-4): finished collecting timing info
2020-10-23 18:10:46.084193 (Thread-4): Sending event: {'category': 'dbt', 'action': 'run_model', 'label': '97e01ffa-fb11-4dbb-905d-f2aa02358b5c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7feef1b60460>]}
2020-10-23 18:10:46.086245 (Thread-4): 11:10:46 | 4 of 4 OK created table model dbt_kristin_marketo.stg_salesforce_user_role [CREATE TABLE (96.0 rows, 4.5 KB processed) in 3.94s]
2020-10-23 18:10:46.086422 (Thread-4): Finished running node model.salesforce_source.stg_salesforce_user_role
2020-10-23 18:10:46.088714 (Thread-3): finished collecting timing info
2020-10-23 18:10:46.089394 (Thread-3): Sending event: {'category': 'dbt', 'action': 'run_model', 'label': '97e01ffa-fb11-4dbb-905d-f2aa02358b5c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7feef1b4b100>]}
2020-10-23 18:10:46.090536 (Thread-3): 11:10:46 | 3 of 4 OK created table model dbt_kristin_marketo.stg_salesforce_user [CREATE TABLE (376.0 rows, 26.0 KB processed) in 3.95s]
2020-10-23 18:10:46.090682 (Thread-3): Finished running node model.salesforce_source.stg_salesforce_user
2020-10-23 18:10:47.141862 (Thread-2): finished collecting timing info
2020-10-23 18:10:47.142715 (Thread-2): Sending event: {'category': 'dbt', 'action': 'run_model', 'label': '97e01ffa-fb11-4dbb-905d-f2aa02358b5c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7feef1b8d070>]}
2020-10-23 18:10:47.144027 (Thread-2): 11:10:47 | 2 of 4 OK created table model dbt_kristin_marketo.stg_salesforce_opportunity [CREATE TABLE (30.0k rows, 4.3 MB processed) in 5.01s]
2020-10-23 18:10:47.144201 (Thread-2): Finished running node model.salesforce_source.stg_salesforce_opportunity
2020-10-23 18:10:48.052159 (Thread-1): finished collecting timing info
2020-10-23 18:10:48.053015 (Thread-1): Sending event: {'category': 'dbt', 'action': 'run_model', 'label': '97e01ffa-fb11-4dbb-905d-f2aa02358b5c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7feef1bec9d0>]}
2020-10-23 18:10:48.054347 (Thread-1): 11:10:48 | 1 of 4 OK created table model dbt_kristin_marketo.stg_salesforce_account [CREATE TABLE (265.4k rows, 27.5 MB processed) in 5.94s]
2020-10-23 18:10:48.054521 (Thread-1): Finished running node model.salesforce_source.stg_salesforce_account
2020-10-23 18:10:48.055947 (MainThread): Acquiring new bigquery connection "master".
2020-10-23 18:10:48.056311 (MainThread): 11:10:48 | 
2020-10-23 18:10:48.056452 (MainThread): 11:10:48 | Finished running 4 table models in 9.76s.
2020-10-23 18:10:48.056570 (MainThread): Connection 'master' was properly closed.
2020-10-23 18:10:48.056672 (MainThread): Connection 'model.salesforce_source.stg_salesforce_account' was properly closed.
2020-10-23 18:10:48.056753 (MainThread): Connection 'model.salesforce_source.stg_salesforce_opportunity' was properly closed.
2020-10-23 18:10:48.056831 (MainThread): Connection 'model.salesforce_source.stg_salesforce_user' was properly closed.
2020-10-23 18:10:48.056906 (MainThread): Connection 'model.salesforce_source.stg_salesforce_user_role' was properly closed.
2020-10-23 18:10:48.070603 (MainThread): 
2020-10-23 18:10:48.070786 (MainThread): Completed successfully
2020-10-23 18:10:48.070919 (MainThread): 
Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
2020-10-23 18:10:48.071137 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7feef19336a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7feef1b2fac0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7feef1b2f640>]}
2020-10-23 18:10:48.071346 (MainThread): Flushing usage events
